<h2>Description</h2>

<p>
This track was produced as part of the mouse ENCODE Project.
It shows DNaseI sensitivity measured genome-wide in mouse
<a href="/cgi-bin/hgEncodeVocab?type=cellType" target="_blank">tissues and cell lines</a>
using the Digital DNaseI methodology (see below), and DNaseI hypersensitive
sites. DNaseI has long been used to map general chromatin accessibility and
DNaseI hypersensitivity is a universal feature of active
<em>cis</em>-regulatory sequences. The use of this method has led
to the discovery of functional regulatory elements that include enhancers,
insulators, promoters, locus control regions and novel elements.
For each experiment (tissue/cell type), this track shows DNaseI sensitivity as a
continuous function using sequencing tag density (<em>Signal</em>),
and discrete loci of DNaseI sensitive zones (<em>HotSpots</em>) and
hypersensitive sites (<em>Peaks</em>).
</p>

<h2>Display Conventions and Configuration</h2>

<p>
This track is a multi-view composite track that contains multiple data types
(<em>views</em>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <a href="/goldenPath/help/multiView.html" target="_blank">here</a>.
This track contains the following views:
</p>

<dl>
<dt><em>HotSpots</em></dt>
<dd> DNaseI sensitive zones identified using the HotSpot algorithm.</dd>
<dt><em>Peaks</em></dt>
<dd> DNaseI hypersensitive sites (DHSs) identified as signal peaks within
FDR 1.0% hypersensitive zones.</dd>
<dt><em>Signal</em></dt>
<dd>The density of tags mapping within a 150 bp sliding window
(at a 20 bp step across the genome).</dd>
</dl>

<p>
DNaseI sensitivity is shown as the absolute density of <em>in vivo</em>
cleavage sites across the genome mapped using the Digital DNaseI methodology
(see below). Data have been normalized to 25 million reads per cell type.
</p>

<p>
Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.
</p>

<H2>Methods</H2>

<p>
Cells were grown according to the approved
<a href="/ENCODE/protocols/cell/mouse" target="_blank">ENCODE cell culture protocols</a>.
Fresh tissues were harvested from mice and the nuclei prepared according
to the tissue-appropriate
<a href="/ENCODE/protocols/cell/mouse" target="_blank">protocol</a>.
Digital DNaseI was performed by DNaseI digestion of intact
nuclei, isolating DNaseI 'double-hit' fragments as described in
Sabo <em>et al</em>. (2006), and direct sequencing of fragment
ends (which correspond to <em>in vivo</em> DNaseI cleavage sites)
using the Illumina IIx (and Illumina HiSeq by early 2011) platform (36 bp reads).
Uniquely-mapping high-quality reads were mapped to the genome using
<a href="http://bowtie-bio.sourceforge.net/index.shtml" target="_blank">Bowtie</a>.
DNaseI sensitivity is directly reflected in
raw tag density, which is shown in the track as density of tags
mapping within a 150 bp sliding window (at a 20 bp step across the genome).
DNaseI sensitive zones (<em>HotSpots</em>) were identified using the
HotSpot algorithm described in Sabo <em>et al</em>. (2004). False discovery
rate thresholds of 1.0% (FDR 1.0%) were computed for each cell type by
applying the HotSpot algorithm to an equivalent number of random
uniquely-mapping 36mers.  DNaseI hypersensitive sites (DHSs or <em>Peaks</em>)
were identified as signal peaks within FDR 1.0% hypersensitive
zones using a peak-finding algorithm (<em>I-max</em>).
</p>

<h2>Verification</h2>

<p>
Data were verified by sequencing biological replicates displaying
correlation coefficient &gt; 0.9.
</p>

<h2>Release Notes</h2>

<p>
This is Release 2 (September 2012) of this track. It adds 32 new experiments including 22 new cell lines and 4 new treatments.
</p>

<H2>Credits</H2>

<p>
These data were generated by the UW ENCODE group.
</p>

<p>
Contact: <a href="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">
Richard Sandstrom</a>
<!-- above address is sull at u.washington.edu -->
</p>

<h2>References</h2>

<p>
John S, Sabo PJ, Thurman RE, Sung MH, Biddie SC, Johnson TA, Hager GL, Stamatoyannopoulos JA.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21258342" target="_blank">
Chromatin accessibility pre-determines glucocorticoid receptor binding patterns</a>.
<em>Nat Genet</em>. 2011 Mar;43(3):264-8.
</p>

<p>
Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J, Hall R, Mack J, Dorschner
MO <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/15550541" target="_blank">
Discovery of functional noncoding elements by digital analysis of chromatin structure</a>.
<em>Proc Natl Acad Sci U S A</em>. 2004 Nov 30;101(48):16837-42.
</p>

<p>
Sabo PJ, Kuehn MS, Thurman R, Johnson BE, Johnson EM, Cao H, Yu M, Rosenzweig E, Goldy J, Haydock A
<em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16791208" target="_blank">
Genome-scale mapping of DNase I sensitivity in vivo using tiling DNA microarrays</a>.
<em>Nat Methods</em>. 2006 Jul;3(7):511-8.
</p>

<h2>Data Release Policy</h2>

<p>
Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <em>Restricted Until</em> column, above.  The full data release policy
for ENCODE is available
<a href="/ENCODE/terms.html" target="_blank">here</a>.
</p>
