<H2>Description</H2>

<P><DT><B><I>Rationale for the Mouse ENCODE project</I></B></DT>
Knowledge of the function of genomic DNA sequences comes from three 
basic approaches. Genetics uses changes in behavior or structure of a cell or organism
in response to changes in DNA sequence to infer function of the altered sequence. 
Biochemical approaches monitor states of histone modification, binding of specific 
transcription factors, accessibility to DNases and other epigenetic features along 
genomic DNA. In general, these are associated with gene activity, but the precise 
relationships remain to be established. The third approach is evolutionary, using 
comparisons among homologous DNA sequences to find segments that are evolving 
more slowly or more rapidly than expected given the local rate of neutral change. These 
are inferred to be under negative or positive selection, respectively, and interpreted 
as DNA sequences needed for a preserved (negative selection) or adaptive 
(positive selection) function. </P>

<P>The ENCODE project aims to discover all the DNA sequences associated with 
various epigenetic features, with the reasonable expectation that these will also be 
functional &#40;best tested by genetic methods&#41;. However, it is not clear how to relate these 
results with those from evolutionary analyses. The mouse ENCODE project aims to 
make this connection explicitly and with a moderate breadth. Assays identical to those 
being used in the ENCODE project are performed in cell types in mouse that are similar 
or homologous to those studied in the human project. The comparison will be used to discover
which epigenetic features are conserved between mouse and human, and  
examine the extent to which these overlap with the DNA sequences under negative 
selection. The contribution of functional DNA  preserved in mammals versus 
function in only one species will be discovered. The results will have a 
significant impact on the understanding of the evolution of gene regulation.
</P>

<P><DT><B><I>Maps of histone modifications</I></B></DT>
Levels of five histone modifications were determined. 
Methylation of lysine 4 of histone H3 is associated with active chromatin, with monomethylation (H3K4me1) 
associated with enhancers and trimethylation (H3K4me3) associated with active promoters. 
Trimethylation of lysine 36 (H3K36me3) is associated with elongating RNA polymerase II. 
Two marks associated with repressed chromatin were also determined, trimethylation of lysine 27 of histone H3 (H3K27me3) which is 
deposited by Polycomb repressive complex 2, and trimethylation of lysine 9 of histone H3 (H3K9me3). 
Maps of genomic DNA in chromatin with these histone modifications are generated by ChIP-seq. 
This consists of two basic steps: chromatin immunoprecipitation (ChIP) is used to highly enrich 
genomic DNA for the segments bound by specific proteins (the antigens recognized by the antibodies) 
followed by massively parallel short read sequencing to tag the enriched DNA segments. Sequencing was done on the Illumina GAIIx and HiSeq. 
The sequence tags are mapped back to the mouse genome (&#40;Langmead <em>et al.</em>, 2009&#41;), 
and a graph of the enrichment for histone modifications are displayed as the "Signal" track (essentially the counts of mapped reads per interval) 
and the deduced probable binding sites from the MACS program (&#40;Zhang <em>et al.</em>, 2008&#41;) are shown in the "Peaks" track. 
Each experiment is associated with an input signal, which represents the control condition where immunoprecipitation with non-specific immunoglobulin was performed in the same cell type. 
The sequence reads, quality scores, and alignment coordinates from these experiments are available for download.
</P>

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <a href="/goldenPath/help/multiView.html" target="_blank">here</a>.
This track contains the following views:
<DL>
<DT><I>Peaks</I></DT><DD> Regions of signal enrichment based on processed data
(usually normalized data from pooled replicates). Intensity is represented in 
grayscale; the darker shading shows higher intensity (a solid vertical line
in the peak region represents the point with the highest signal).
ENCODE Peaks tables contain fields for statistical significance, including FDR
(<a href="/goldenPath/help/qValue.html" target="_blank">qValue</a>).</DD>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based on
processed data.</DD>
</DL>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<P>
Cells were grown according to the approved
<a href="/ENCODE/protocols/cell/mouse" target="_blank">ENCODE cell culture protocols</a>.
</P>
<P>
The chromatin immunoprecipitation followed published methods &#40;Welch <EM>et al.</EM>, 2004&#41;.
Information on antibodies used is available via the hyperlinks in the &#34;Select
subtracks&#34; menu. Samples passing initial quality thresholds &#40;enrichment and depletion 
for positive and negative controls &#45; if available &#45; by quantitative PCR of ChIP material&#41;
are processed for library construction for Illumina sequencing, using the ChIP-seq 
Sample Preparation Kit purchased from Illumina. Starting with a 10 ng sample of ChIP 
DNA, DNA fragments were repaired to generate blunt ends and a single A nucleotide 
was added to each end. Double-stranded Illumina adaptors were ligated to the 
fragments. Ligation products were amplified by 18 cycles of PCR, and the DNA between 
250-350 bp was gel purified. Completed libraries were quantified with Quant-iT dsDNA 
HS Assay Kit. The DNA library was sequenced on the Illumina Genome Analyzer II 
sequencing system, and more recently on the HiSeq. Cluster generation, linearization, 
blocking and sequencing primer reagents were provided in the Illumina Cluster 
Amplification kits. All samples were determined as biological replicates except time 
course samples. The data displayed are from the pooled reads for all replicates, but 
individual replicates are available by download.
</P>

<P>
The resulting 36-nucleotide sequence reads &#40;fastq files&#41; were moved to a data 
library in Galaxy, and the tools implemented in Galaxy were used for further processing 
via workflows &#40;Blankenberg <EM>et al.</EM>, 2010&#41;. The reads were mapped to the mouse genome 
&#40;mm9 assembly&#41; using the program bowtie &#40;Langmead <EM>et al.</EM>, 2009&#41;, and the files of 
mapped reads for the ChIP sample and from the &#34;input&#34; control &#40;no antibody&#41; were 
processed by MACs &#40;Zhang <EM>et al.</EM>, 2008&#41; to call peaks for occupancy by transcription 
factors, using the parameters mfold=15, bandwidth=125. Since, the signal for some histone 
modifications is not expected to be tightly localized &#40;compared to a transcription factor&#41;,
peak calling programs may not be appropriate. Thus in addition, we provide wiggle 
tracks with tag counts for every 10 bp segment. Per-replicate
alignments and sequences are available for download at
<A HREF="../cgi-bin/hgFileUi?db=mm9&amp;g=wgEncodePsuHistone"
 TARGET=_BLANK>downloads page</A>.
</P>

<h2>Release Notes</h2>

<p>
This is Release 2 (August 2012). It contains a total of 30 ChIP-seq experiments on Histone Modifications with the addition of 1 new
experiment.
</p>
<p>
Previous versions of files are available for download from the
<a HREF="ftp://hgdownload.cse.ucsc.edu/goldenPath/mm9/encodeDCC/wgEncodePsuHistone/"
TARGET=_BLANK>FTP site</A>.
</p>


<H2>Credits</H2>

<p>
Cell growth, ChIP, and Illumina library construction were done primarily by Weisheng Wu, 
and sequencing on the Illumina platform was done largely by Cheryl Keller in the laboratory of 
Ross Hardison (PSU). Data processing and analysis were overseen by James Taylor (Emory University), 
using tools provided in the Galaxy platform (Anton Nekrutenko, PSU, and James Taylor, Emory) enabled 
by the Penn State Cyberstar computer (supported by the National Science Foundation). 
Generation of these data was supported by National Institutes of Health grants R01DK065806 and RC2HG005573.
</p>
<p>
Contact:
<A HREF="mailto:&#114;&#99;h&#56;&#64;&#112;&#115;&#117;.
&#101;&#100;u"
title="mailto:&#114;&#99;h&#56;&#64;&#112;&#115;&#117;.
&#101;&#100;u" rel="nofollow"
 TARGET=_BLANK>Ross Hardison</A>
<!-- above address is Ross Hardison at psu.edu -->
</p>

<H2>References</H2>

<p>
Bernstein BE, Mikkelsen TS, Xie X, Kamal M, Huebert DJ, Cuff J, Fry B, Meissner A, Wernig M, Plath K <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16630819" target="_blank">
A bivalent chromatin structure marks key developmental genes in embryonic stem cells</a>.
<em>Cell</em>. 2006 Apr 21;125(2):315-26.
</p>

<p>
Blankenberg D, Gordon A, Von Kuster G, Coraor N, Taylor J, Nekrutenko A, Galaxy Team.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20562416" target="_blank">
Manipulation of FASTQ data with Galaxy</a>.
<em>Bioinformatics</em>. 2010 Jul 15;26(14):1783-5.
</p>

<p>
ENCODE Project Consortium, Birney E, Stamatoyannopoulos JA, Dutta A, Guig&#243; R, Gingeras TR, Margulies EH, Weng Z, Snyder M, Dermitzakis ET <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17571346" target="_blank">
Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</a>.
<em>Nature</em>. 2007 Jun 14;447(7146):799-816.
</p>

<p>
Heintzman ND, Stuart RK, Hon G, Fu Y, Ching CW, Hawkins RD, Barrera LO, Van Calcar S, Qu C, Ching KA <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17277777" target="_blank">
Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome</a>.
<em>Nat Genet</em>. 2007 Mar;39(3):311-8.
</p>

<p>
Langmead B, Trapnell C, Pop M, Salzberg SL.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" target="_blank">
Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</a>.
<em>Genome Biol</em>. 2009;10(3):R25.
</p>

<p>
M&#252;ller J, Hart CM, Francis NJ, Vargas ML, Sengupta A, Wild B, Miller EL, O'Connor MB, Kingston RE, Simon JA.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/12408864" target="_blank">
Histone methyltransferase activity of a Drosophila Polycomb group repressor complex</a>.
<em>Cell</em>. 2002 Oct 18;111(2):197-208.
</p>

<p>
Rando OJ, Chang HY.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19317649" target="_blank">
Genome-wide views of chromatin structure</a>.
<em>Annu Rev Biochem</em>. 2009;78:245-71.
</p>

<p>
Weiss MJ, Yu C, Orkin SH.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/9032291" target="_blank">
Erythroid-cell-specific properties of transcription factor GATA-1 revealed by phenotypic rescue of a gene-targeted cell line</a>.
<em>Mol Cell Biol</em>. 1997 Mar;17(3):1642-51.
</p>

<p>
Welch JJ, Watts JA, Vakoc CR, Yao Y, Wang H, Hardison RC, Blobel GA, Chodosh LA, Weiss MJ.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/15297311" target="_blank">
Global regulation of erythroid gene expression by transcription factor GATA-1</a>.
<em>Blood</em>. 2004 Nov 15;104(10):3136-47.
</p>

<p>
Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, Nusbaum C, Myers RM, Brown M, Li W <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18798982" target="_blank">
Model-based analysis of ChIP-Seq (MACS)</a>.
<em>Genome Biol</em>. 2008;9(9):R137.
</p>

<h2>Publications</h2>

<p>
Wu W, Cheng Y, Keller CA, Ernst J, Kumar SA, Mishra T, Morrissey C, Dorman CM, Chen KB, Drautz D <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21795386" target="_blank">
Dynamics of the epigenetic landscape during erythroid differentiation after GATA1 restoration</a>.
<em>Genome Res</em>. 2011 Oct;21(10):1659-71.
</p>

<H2>Data Release Policy</H2>

<P>
Data users may freely use ENCODE data, but may not, without prior consent, 
submit publications that use an unpublished ENCODE dataset until nine months 
following the release of the dataset. This date is listed in the <em>Restricted Until</em> column, 
above. The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
