<H2>Description</H2>
<P>

This track displays human tissue microarray data using <A HREF="http://www.affymetrix.com/estore/browse/products.jsp?navMode=34000&productId=131452&navAction=jump&aId=productsNav#1_1" 
TARGET=_blank>Affymetrix Human Exon 1.0</A> 
ST expression arrays. This RNA expression track was produced as part of the
ENCODE Project. The RNA was extracted from cells that were also analyzed by DNaseI hypersensitivity (<a href="/cgi-bin/hgTrackUi?g=wgEncodeOpenChromDnase" TARGET=_BLANK>Duke DNaseI HS</A>), FAIRE (<a href="/cgi-bin/hgTrackUi?g=wgEncodeOpenChromFaire" TARGET=_BLANK>UNC FAIRE</A>), and ChIP (<a href="/cgi-bin/hgTrackUi?g=wgEncodeOpenChromChip" TARGET=_BLANK>UTA TFBS</A>).
</P>

<H2>Display Conventions and Configuration</H2>
<P>
In contrast to the hg18 annotation, this track now displays exon array data
that has been aggregated to the gene level for those probes that have been
linked to genes. Probes not linked to genes are not included.
The display for this track shows gene probe location and signal value as
grayscale-colored items where higher signal values correspond to darker-colored
blocks.
</P>

<P>
Items with scores between 900-1000 have signal values greater than 9 that have been linearly scaled for that particular cell type. 
Items scoring 400-900 have signal values between 4 and 9, and the signal is simply multiplied by 100 to get the score. 
Items with scores between 200-400 have signal values below 4 that have been linearly scaled to fit that score range. 
</P>
<P>
The subtracks within this composite annotation track correspond to data from different
cell types and tissues. The configuration options are shown at the top of the track
description page, followed by a list of subtracks. To display only selected subtracks,
uncheck the boxes next to the tracks you wish to hide.
</P>
<P>
For information regarding specific microarray probes, turn on the <a
href="/cgi-bin/hgTrackUi?g=affyExonArray" TARGET=_BLANK>Affy Exon Probes</A> track, which 
can be found in the <B>Expression</B> track group.  See Methods for a description
as to how probe level data was processed to produce gene level annotations.
</P>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<p>
Data from these tracks are stored as bed files whose first six fields follow the bed file standard. The three additional fields are as follows:
<ul>
<li>signalValue: The normalized expression value for a gene, calculated as described below.</li>
<li>exonCount: The number of exons used in the calculation of the expression value.</li>
<li>constitutiveExons: The number of constitutive exons used in the calculation of the expression value.</li>
</ul>

<H2>Methods</H2>
<P>

Cells were grown according to the approved
 <A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Total RNA was isolated from these cells using trizol extraction followed by
cleanup on RNEasy column (Qiagen) that included a DNaseI step. 
The RNA was checked for quality using a nanodrop and an Agilent Bioanalyzer.
RNA (1 &micro;g) deemed to be of good quality was then processed
either by 1) the standard Affymetrix Whole transcript Sense Target labeling protocol that included a riboreduction step,
or 2) the NuGEN labeling system.
The fragmented biotin-labeled cDNA was hybridized over 16 h to Affymetrix Exon 1.0 ST arrays and scanned on an Affymetrix Scanner 3000 7G using AGCC software. 
</P>
<P>
Data from all replicates were then normalized together.
Probesets flagged as cross-hybridizing were removed from the analysis (Salomonis <em>et al.</em> 2010).
Though these arrays provide exon-level resolution, gene-level expression was estimated by
grouping probesets by gene for normalization (Bemmo <em>et al.</em> 2008). Probesets were assigned
to genes based on the GENCODE v10 annotation (July 2011). An exon was classified as
constitutive or non-constitutive based on whether it was present in all protein-coding transcripts.
For genes with at least 4 constitutive probes, only constitutive probesets were used to
estimate gene expression. For all other genes, including all non-protein-coding genes, 
all (non-cross-hybridizing) probesets that mapped to an expressed exon in any transcript of the gene were used. 
Gene-level expression estimates were normalized using Affymetrix Power Tools (APT) (Lockstone 2011)
with the chipstream command "rma-bg, med-norm, pm-gcbg, med-polish". This chipstream calls 
for an RMA normalization with gc-background correction using antigenomic background probes. 
</P>
<p>
While the data was generated using the same microarray platform, two different experimental
backgrounds were present due to a change in labeling reagents (Affymetrix vs. NuGEN; see Methods above). 
It was found that batch effects related to this change were causing array data to group by experimental protocol
rather than cell type relatedness. We used an R script (ComBat) to correct for this batch effect (Johnson <em>et al.</em> 2007).
<p>

<H2>Verification</H2>
<P>
When biological replicates were available, data were verified by analyzing replicates displaying a Pearson correlation coefficient &gt; 0.9.
</P>

<H2>Release Notes</H2>
<P>
This is release 3 of this track (April 2012). Several new cell types have been added. The name of cell line Astrocy was changed to NH-A. 
</P>

<H2>Credits</H2>

<P>
RNA was extracted from each cell type by Greg Crawford's group at 
<a href="http://www.genome.duke.edu/labs/crawford/" target=_blank>Duke University</a>.
RNA was purified and hybridized to Affymetrix Exon arrays by <a href="http://www.albany.edu/genomics/bios/bio-chittur.html" target="_blank">Sridar Chittur</a> and
<a href="http://www.albany.edu/cancergenomics/faculty/stenenbaum/index.shtml" target="_blank">Scott Tenenbaum</a> at the University of Albany-SUNY. 
Data analyses were primarily performed by <a href="http://www.genome.duke.edu/labs/crawford/members/nathan_sheffield/" target="_blank">Nathan Sheffield</a> (Duke University) with assistance from <a href="http://www.soe.ucsc.edu/people/cline" target="_blank">Melissa Cline</a> (UCSC), <a href="http://www.unc.edu/~zczhang/" target="blank">Zhancheng Zhang</a> (UNC Chapel Hill), and <a href="http://www.genome.duke.edu/cores/computational/contact/" target="_blank">Darin London</a> (Duke University). 
</P>

<p>Contact: 
<A HREF="mailto:&#116;&#115;&#102;ur&#101;&#121;&#64;&#101;&#109;a&#105;&#108;.
&#117;&#110;c.
e&#100;&#117;">
Terry Furey
</A>
<!-- above address is tsfurey at email.unc.edu -->

<H2>References</H2>

<p>
Bemmo A, Benovoy D, Kwan T, Gaffney DJ, Jensen RV, Majewski J.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990248" target="_blank">
Gene expression and isoform variation analysis using Affymetrix Exon Arrays</a>.
<em>BMC Genomics</em>. 2008 Nov 7;9:529.
</p>

<p>
Johnson WE, Li C, Rabinovic A.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16632515" target="_blank">
Adjusting batch effects in microarray expression data using empirical Bayes methods</a>.
<em>Biostatistics</em>. 2007 Jan;8(1):118-27.
</p>

<p>
Lockstone HE.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21498550" target="_blank">
Exon array data analysis using Affymetrix power tools and R statistical software</a>.
<em>Brief Bioinform</em>. 2011 Nov;12(6):634-44.
</p>

<p>
Salomonis N, Schlieve CR, Pereira L, Wahlquist C, Colas A, Zambon AC, Vranizan K, Spindler MJ, Pico AR, Cline MS <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20498046" target="_blank">
Alternative splicing regulates mouse embryonic stem cell pluripotency and differentiation</a>.
<em>Proc Natl Acad Sci U S A</em>. 2010 Jun 8;107(23):10514-9.
</p>

<H2>Data Release Policy</H2> 
<P>
Data users may freely use ENCODE data, but
may not, without prior consent, submit publications that use an
unpublished ENCODE dataset until nine months following the release of
the dataset. This date is listed in the Restricted Until column,
above. The full data release policy for ENCODE is available
<A href="/ENCODE/terms.html">here</A>.
</P>
