<H2>Description</H2>
<P>This track, produced as part of the ENCODE Project,
displays maps of histone modifications genome-wide using ChIP-seq in different
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>.
The ChIP-seq method involves first using formaldehyde
to cross-link histones and other DNA-associated proteins to genomic DNA 
within cells. The cross-linked chromatin is subsequently extracted, 
sheared, and immunoprecipitated using specific antibodies. 
After reversal of cross-links, the immunoprecipitated DNA is
sequenced and mapped to the human reference genome. The relative 
enrichment of each antibody-target (epitope) across the genome is 
inferred from the density of mapped fragments.
</P>
    
<P>Chemical modifications (e.g. methylation or acetylation) of 
the histone proteins present in chromatin influence gene expression
by changing how accessible the chromatin is to transcription factors.
Shown for each
experiment (defined as a particular antibody and a particular
cell type) is a track of enrichment for the specifically modified
histone (<I>Signal</I>), along with sites that have the
greatest enrichment (<I>Peaks</I>).  Also, included for each
cell type is the input signal, which represents the
control condition where no antibody targeting was performed.
In general, the following chemical modifications have associated genetic
phenotypes:
</P>
<ul>
<li>H3K4me3 and H3K9ac are considered to be marks of active
or potentially active promoter regions </li>
<li>H3K4me1 and H3K27ac are
considered to be marks of active or potentially active enhancer
regions\ </li>
<li>H3K36me3 and H3K79me2 are considered to be marks of
transcriptional elongation</li>
<li>H3K27me3 and H3K9me3 are
considered to be marks of inactive regions.</li>
</ul>

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
</P>
<P>
For each cell type, this track contains the following views:
<DL>
<DT><I>Peaks</I></DT>
<DD>Regions of signal enrichment based on processed data (usually normalized data from pooled replicates).</DD>
<DT><I>Signal</I></DT>
<DD>Density graph (wiggle) of signal enrichment based on aligned read density.</DD>
</DL>
</P>
<P>Peaks and signals displayed in this track are the results of pooled replicate
sequence. Alignment files for each replicate are available for
<A HREF="/cgi-bin/hgFileUi?g=wgEncodeSydhHistone">download</A>.
</P>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<P> 
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Briefly, cells were cross-linked, chromatin was extracted and sonicated
using a Bioruptor sonicator (Diagenode) to an average size of 300-500 bp,
and individual ChIP assays were performed using antibodies to modified histones.
For the K562, MCF-7, HCT-116, NTera-2 (NT2-D1), PANC-1 and PBMC histone ChIP-seq samples, immunoprecipitates were 
collected using protein G-coupled magnetic beads; a detailed
<A HREF="http://www.roadmapepigenomics.org/protocols" TARGET=_BLANK> ChIP and library
protocol</A>
can be found at the Roadmap Epigenome Project.
For the U2OS histone ChIP-seq samples, immunoprecipitates were collected using 
StaphA cells; a 
<A HREF="http://expression.genomecenter.ucdavis.edu/chip.html" TARGET=_BLANK>detailed protocol</A>
can be found at the UCD Genome Center.
Library DNA was quantitated using either a Nanodrop or a BioAnalyzer and sequenced on an Illumina GA2.  
</P>

<P> 
The sequencing reads were mapped to the genome using the Eland alignment program.
ChIP-seq data was scored based on sequence reads (length &#126;30 bps) that align uniquely
to the human genome. From the mapped tags, a signal map of ChIP DNA fragments
(average fragment length &#126; 200 bp) was constructed where the 
signal height is the number of overlapping fragments at each nucleotide position in the genome. 
</P>

<P> 
For each 1 Mb segment of each chromosome, a peak height threshold was determined by
requiring a false discovery rate &lt;&#61; 0.05 when comparing the number of peaks 
above threshold as compared to the number obtained from multiple simulations of 
a random null background with the same number of mapped reads
(also accounting for the fraction of mapable bases for sequence tags in that 1 Mb segment).
The number of mapped tags in a putative binding region is compared to the
normalized (normalized by correlating tag counts in genomic 10 kb windows) 
number of mapped tags in the same region from an input DNA control. 
Using a binomial test, only regions that have a p-value &lt;&#61; 0.05 are considered 
to be significantly enriched compared to the input DNA control. 
</P>

<h2>Release Notes</h2>
<p>
This is Release 3 (June 2012) of this track, which adds 9 new experiments for the MCF-7, HCT-116 and PANC-1 cell lines. 
</p>


<H2>Credits</H2>

<P> 
These data were generated and analyzed by the labs of Peggy Farnham
(USC/Norris Cancer Center; previously at UC Davis) and Michael Snyder
at Stanford University.
</P>
<P>
Contact:
<A HREF="mailto:&#112;&#102;a&#114;&#110;h&#97;&#109;&#64;&#117;&#115;&#99;.
&#101;&#100;u">Peggy Farnham</A>
 <!-- above address is pfarnham at usc.edu -->
 for questions concerning data collection and usage 
and <A HREF="mailto:&#112;&#99;&#97;y&#116;&#105;&#110;g&#64;&#115;&#116;&#97;&#110;&#102;&#111;rd.
&#101;&#100;u">
Philip Cayting</A>
<!-- above address is pcayting at stanford.edu -->
for data scoring and submission inquiries.
</P>

<H2>References</H2>


<P>
Blahnik KR, Dou L, Echupare L, Iyengar S, O&#39;Geen H, et al.
<A HREF ="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3039671/"
        class = "external text"
        title="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3039671/"
        TARGET =" _BLANK">Characterization of the Contradictory Chromatin Signatures at the 3&#39; Exons of Zinc Finger Genes</A>.
<EM>PLoS One.</EM> 2011;6(2):e17121.
</P>
<P>
O&#39;Geen H, Echipare L, Farnham PJ
    <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21913086"
	class="external text"
	title="http://www.ncbi.nlm.nih.gov/pubmed/21913086"
	TARGET =" _BLANK">
	Using ChIP-seq technology to generate high-resolution profiles of histone modifications</A>.
 <EM>Methods Mol Biol.</EM> 2011;791;265-286.
</P>

<P>
O&#39;Geen H, Frietze S, Farnham PJ 
	<A HREF="http://www.ncbi.nlm.nih.gov/pubmed/20680851?dopt=abstract&otool=uscnmlib"
	class="external text"
	title="http://www.ncbi.nlm.nih.gov/pubmed/20680851?dopt=abstract&otool=uscnmlib"
	TARGET = "_BLANK">Using ChIP-seq Technology to Identify Targets of Zinc Finger Transcription Factors</A>.
 <EM>Methods Mol Biol.</EM> 2010;649:437-455.
</P>



<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET=_BLANK HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>

