<H2>Description</H2>

<P>This track is produced as part of the ENCODE Transcriptome Project.
It shows high throughput sequencing of 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=rnaExtract">
RNA samples</A> from 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=localization">
tissues or sub cellular compartments</A> from 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>
included in the ENCODE Transcriptome subproject. The overall 
goal of the ENCODE project is to identify and characterize all functional 
elements in the sequence of the human genome.
</P>

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
<P>To show only selected subtracks, uncheck the boxes next to the tracks that 
you wish to hide.</P>
 
<P>Color differences among the views are arbitrary. They provide a 
visual cue for 
distinguishing between the different cell types.</P>

<DL>
<DT><i>Plus Raw Signal</i>
<DD>The Plus Raw Signal view graphs the base-by-base density of alignments 
on the + strand.
<DT><i>Minus Raw Signal</i>
<DD>The Minus Raw Signal view graphs the base-by-base density of alignments
on the - strand.
<DT><i>Alignments</i>
<DD>The Alignments view shows reads mapped to the genome, both split alignments and alignments mapped to one exon.  Sequences determined to be transcribed on the positive strand are shown in <span style="color:blue">blue</span>.  
Sequences determined to be transcribed on the negative strand are shown in 
<span style="color:red">red</span>.  Sequences for which the direction of 
transcription was not able to be determined are shown in <B>black</B>. 
For more information on the XL XJ and XU custom tags used in these files, please contact the producing lab. Please see the <a href="http://bowtie-bio.sourceforge.net/manual.shtml#sam-bowtie-output" target=_blank>Bowtie Manual</a> for more information about the SAM Bowtie output (including other tags) and the 
<a href="http://samtools.sourceforge.net/SAM1.pdf#page=4" target=_blank>SAM Format Specification</a> 
for more information on the SAM/BAM file format.
</DL>

<H2>Methods</H2>

<P>
The RNA-Seq data were generated from high quality polyA RNA, and the RNA-Seq
libraries were constructed using SOLiD Whole Transcriptome (WT) protocol and
reagent kit. Total RNA in good quality was used as starting materials and
purified twice through MACs polyT column aimed to enrich polyA and remove any
contaminants (e.g., rRNA, tRNA, DNA, protein etc.). A one microgram enriched
polyA RNA sample was then fragmented to small pieces, and a gel-based selection
method was performed to collect fragmented random polyA at a size-range of
50-150 nt in length. The collected fragmental RNA was then hybridized and
ligated to a mix of adapters provided from ABI, followed by reverse
transcription to generate corresponding cDNAs. The resulting cDNA library was
further amplified by PCR and sequenced by SOLiD platform for single reads at 35
bp length (new version in 50 bp length).    
Cells were grown according to the approved 
<A HREF="/ENCODE/protocols/cell/human/" TARGET=_BLANK>ENCODE cell culture
protocols</A>.
</P>

<P>
<B>Data:</B> The SOLiD-generated RNA-Seq reads were 35 bp in length. An 
initial filtering process was performed to remove any non-desirable contamination
sequences, such as rRNA, tRNA, and repeats etc. A read-split mapping approach
was developed to map the 35 bp reads onto the reference genome
(GRCh37/hg19) excluding mitochondrion, haplotypes, randoms and chromosome Y.
</P>

<P>
<B>Mapping parameters:</B>  Strand specific mapping was done using Applied Biosystems' SOLiD 
alignment where all the reads were mapped to the genome, and to exon-exon junction database.  Seed and extend strategy is adopted where initial seed length of 25 is mapped with maximum of 2 mismatches and then extended to read length, each color space match is awarded a score of +1 and each mismatch is awarded a penalty of -2.
</P>

<P>
After extension each read is trimmed to its maximum score, shortest length.
The color space sequences are then converted into base space and checked to ensure that each sequence has a maximum of 2 base pair mismatches.  If any sequence has more than 2 mismatches, then that sequence is discarded.
</P>

<H2>Credits</H2>

<P>The GIS RNA-seq libraries and sequence data for transcriptome analysis were
produced at the 
<A HREF="http://www.gis.a-star.edu.sg/internet/site">Genome Institute of
Singapore</A>. 
The data were mapped and analyzed by scientists from the Genome Institute of 
Singapore.
</P>

<P>Contact: <A HREF="mailto:&#114;&#117;a&#110;&#120;&#64;&#103;&#105;&#115;.
a&#45;&#115;t&#97;&#114;.
&#101;&#100;&#117;.
s&#103;">
RUAN Xiaoan</A>
<!-- above address is ruanx at gis.a-star.edu.sg -->

</P>

<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior 
consent, submit publications that use an unpublished ENCODE dataset until 
nine months following the release of the dataset.  This date is listed in 
the <EM>Restricted Until</EM> column, above.  The full data release policy 
for ENCODE is available 
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P> 
