<H2>Description</H2>
<P>
This track shows 5' cap analysis gene expression (CAGE) tags and clusters in 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=rnaExtract">RNA extracts</A> 
from different 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=localization">
sub-cellular localizations</A> 
in multiple
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>.
A CAGE cluster is a region of overlapping tags with
an assigned value that represents the expression level.
The data in this track were produced as part of the ENCODE 
Transcriptome Project. 

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are 
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
To show only selected subtracks, uncheck the boxes next to the tracks that 
you wish to hide.</P>

<P>This track contains the following views:
<DL>
<!--<DT><I>Clusters</I> </DT>
<DD> Pooled clusters generated by merging clusters from individual bioreplicates using mergedBed 
(where two bioreplicates were available). Expression was scored individually for each replicate. 
The <em>score</em> represents the average expression score of the pooled replicates.
Non-parametric Irreproducible Discovery Rate (np-IDR) was run on the normalized expression scores. 
The expression score is number of reads mapping to a merged 
cluster divided by the number of total mapped reads. A complete 
description of the Cluster files is located in the <A TARGET=_BLANK 
HREF="http://hgdownload-test.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeRikenCage/supplemental/">supplemental 
materials directory</A> of our <A TARGET=_BLANK HREF="../../cgi-bin/hgFileUi?db=hg19&g=wgEncodeRikenCage">downloads page</A>.
-->

<DT><I>TSS HMM</I></DT>
<DD> Transcriptional Start Sites based on Hidden Markov Modeling for pooled replicates where two replicates exist. 
<ul>
<li>Expression levels are shown in reads per kilobase of exon per million reads mapped (RPKM).</li>
<li>The IDR value is the irreproducible discovery rate. This is a measurement that measures expression variances between genomic replicates in large scale experiments.</li>
</ul>
</DD>
<DT><I>Plus</I> and <I>Minus</I> <I>Signals</I>
<DD>These views display signals representing the amount of overlapping CAGE reads (clusters) mapped on the forward and reverse genomic strands.
</DD>
<DT><i>Alignments</i></DT>
<DD>The Alignments view shows reads mapped to the genome and indicates where
bases may mismatch. Every mapped read is displayed, i.e. uncollapsed. 
The alignment file follows the standard SAM format of Bowtie output. The custom tag XP can be ignored. See the <a href="http://bowtie-bio.sourceforge.net/manual.shtml#sam-bowtie-output" target=_blank>Bowtie Manual</a> for more information about the SAM Bowtie output (including other tags) and the <a href="http://samtools.sourceforge.net/SAM1.pdf#page=4" target=_blank>SAM Format Specification</a> for more information on the SAM/BAM file format.
 Where mapping quality is not available for this track, a score of 255 is used in accordance with the SAM Format Specification. Also, where the sequence quality scores are not available, all scores are displayed as 40.
</dd>
</DL>
<P>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</P>
<P> Replicate numbering in the track display page is done by rank.  The first replicate available may be replicate number three.</P>

<P>Color differences in subtracks may be set as a visual cue to
distinguish between the different cell types or between annotations
on the plus and minus strand.
</P>

<h3>Downloadable Files</h3>
<DL>
<DT><I>TSS GencV7</I> </DT>
<DD> For some samples, there are download files in a modified gtf format with Transcriptional Start Sites based on GENCODE V7. A complete description of the TSS files is located in the <A TARGET=_BLANK HREF="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeRikenCage/supplemental/">supplemental materials directory</A>.
</DD>
</dl>

<H2>Methods</H2>
<P>
Cells were grown according to the approved 
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>
ENCODE cell culture protocols</A>.
RNA molecules longer than 200 nt were isolated 
from each subcellular compartment and then
were fractionated into polyA+ and polyA- fractions as described in 
<A HREF="/ENCODE/protocols/general/rnaExtracts.txt" TARGET=_BLANK>these 
protocols</A>.
The CAGE tags were sequenced from the 5' ends of cap-trapped cDNAs produced
using RIKEN CAGE technology 
(Kodzius <EM>et al</EM>. 2006; Valen <EM>et al</EM>. 2009). 
To create the tag, a linker was attached to the 5'
end of polyA+ or polyA- reverse-transcribed cDNAs which were selected by cap
trapping (Carninci <EM>et al</EM>. 1996). The first 27 bp of
the cDNA were cleaved using class II restriction enzymes. A linker was then
attached to the 3' end of the cDNA. 
</P>
<P>
After PCR amplification, the tags were sequenced using Illumina's Genome analyzer or HiSeq. The read lengths for each sample 
are specified in the metadata.
Tags were mapped to the human genome (hg19) using the program Delve (T. Lassmann manuscript in preparation).
Delve is a new probabilistic aligner focused on giving the best possible alignment of reads to a genome rather
than focusing on speed. It calculates the mapping accuracy (probability of each alignment being true or not)
for each alignment. There is no set limit on the number of errors allowed and therefore the mapping rate is
commonly 100%. However, for analysis it is recommended to discard alignments with low mapping qualities.
</P>
<P>
Exceptions to the above protocol are the polyA- RNA samples from K562 cytosol, K562 nucleus, and prostate whole cell
which were sequenced using ABI <A
HREF="http://www.appliedbiosystems.com/absite/us/en/home/applications-technologies/solid-next-generation-sequencing.html"
TARGET=_blank>SOLiD</A>  technology. These reads were mapped using Bowtie with its default parameters.
 Clusters were defined as regions of overlapping CAGE reads. The expression level was computed as the number
 of reads making up the cluster, divided by the total number of reads sequenced, times 1 million.
</P>

<H2>Release Notes</H2>
<P>
This is Release 4 (July 2012) of Riken CAGE. Three missing Transcription Start Sites determined by Hidden Markov Models
(TssHmm) tables have been added (H1-hesc Nucleus, H1-hesc Cytosol and NHEK Nucleus polyA+ samples).
</P>
<P>
 As with previous releases, the orignal data from the hg18 version of this track is still included and can be noted in the metadata as having a bioRepId that starts with gen0. This older data may be missing some information and does not have replicates. If there are new data available for an older sample, only the newer data is displayed.  The older data is still availble for downloads.</P>
<!---
Release 2 has three new downloads only files per experiment (Clusters, TSS Gencode 7 and TSS HMM) and four new cell lines (A459, AG04450, BJ and SK-N-SH_RA).
</P>
<P>
Release 1 on hg19 contained 
the original <A TARGET=_BLANK HREF="/cgi-bin/hgTrackUi?db=hg18&g=wgEncodeRikenCage"> data on hg18</A> 
that was remapped and indicated in this release as Generation 0 since that data had no replicates. 
If there is both old and new generation data available for a particular experiment, only the new generation 
data is displayed and the older data is available for download.  The new data for this track was done 
with a different process and has standard replicate numbers. The replicate labeling in the genome browser view is a
counter indicating the total number of replicates submitted.  The producing lab has replicate numbers 
that correspond to their internal bio-replicate numbering. Where these two numbering systems conflict,
both are listed in the long label of the specific track.
</P>
--->

<H2>Credits</H2>
<P>
These data were generated and analyzed by Timo Lassmann, Phil Kapranov, 
Hazuki Takahashi, Yoshihide Hayashizaki, Carrie Davis, Tom Gingeras, and Piero Carninci.
<P>
<P>Contact: <A HREF="mailto:&#99;&#97;&#114;&#110;i&#110;&#99;i&#64;&#114;&#105;k&#101;&#110;.&#106;&#112;">
Piero Carninci</A> at <A TARGET=_BLANK HREF="http://www.osc.riken.jp/english/">
RIKEN Omics Science Center</A>
<!-- above address is carninci at riken.jp -->
</P>

<H2>References</H2>
<P>
Carninci P, Kvam C, Kitamura A, Ohsumi T, Okazaki Y, Itoh M, Kamiya M, 
Shibata K, Sasaki N, Izawa M, <EM>et al</EM>.
<A TARGET=_BLANK HREF="http://www.ncbi.nlm.nih.gov/pubmed/8938445">
High-efficiency full-length cDNA cloning by biotinylated CAP trapper</A>.
<EM>Genomics</EM>. 1996 November 1; 37(3):327-336.
</P>
<P>
Kodzius R, Kojima M, Nishiyori H, Nakamura M, Fukuda S, Tagami M, Sasaki D,
Imamura K, Kai C, Harbers M, <EM>et al</EM>. 
<A TARGET=_BLANK HREF="http://www.ncbi.nlm.nih.gov/pubmed/16489339">
CAGE: cap analysis of gene expression</A>.
<EM>Nat Methods</EM>. 2006 March 1; 3(3):211-222.
</P>

<P>
Valen E, Pascarella G, Chalk A, Maeda N, Kojima M, Kawazu C, Murata M, Nishiyori
H, Lazarevic D, Motti D, <EM>et al</EM>.
<A TARGET=_BLANK HREF="http://www.ncbi.nlm.nih.gov/pubmed/19074369">
Genome-wide detection and analysis of hippocampus core promoters using
DeepCAGE</A>.
<EM>Genome Res</EM>. 2009 February; 19(2):255-265.
</P>

<H2> Data Release Policy </H2>

<P> Data users may freely use ENCODE data, but may not, without prior 
consent, submit publications that use an unpublished ENCODE dataset until 
nine months following the release of the dataset.  This date is listed in 
the <EM>Restricted Until</EM> column, above.  The full data release policy 
for ENCODE is available 
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P> 

