<H2>Description</H2>

<P>Nucleosomes are part of the first level of chromatin packaging. They each consist of a histone heterooctamer around which DNA wraps 1.6 times. The histone heterooctomamer is made up of two copies of histones 2A, 2B, 3 and 5. The segment of DNA wrapped around the histones, the so-called &quot;core&quot; fragment, is 147 base pairs long. Neighboring nucleosomes are separated from one another by a stretch of DNA called the &quot;linker,&quot; whose size varies depending on organism, cell type, and even chromatin activity. Certain chromatin remodeling factors govern accessibility of DNA to regulatory proteins by repositioning nucleosomes to reveal regulatory sites that would otherwise be occluded by a nucleosome.
</P>

<P>In contrast to histone modifications such as methylation or acetylation, which are investigated by ChIP-seq, nucleosome positioning data are generated without immunoprecipitation (see Methods below).  Instead, micrococcal nuclease is used to digest chromatin to apparent completion, the (well-defined and clearly visible) mononucleosomal core fragment fraction is isolated by gel purification, and one end is then sequenced. Mapping the sequence tag back to the genome reveals the precise position of one end of the core fragment that was protected by the nucleosome;  the position of the other end can then simply be inferred by extending the read to a virtual length of 147 bases. Statistical analyses such as occupancy and positioning stringency can then be employed to analyze the local nucleosome landscape anywhere in the (mappable) genome or to infer global parameters of nucleosome organization.
</P>

<P>In the context of the ENCODE project, nucleosome positioning data are particularly valuable for analysis of the relationship between transcription factor binding, histone modifications, and gene activity.
</P>

<P>For a general primer on these types of data and analyses, refer to Valouev et al. (2008).
</P>

<H2>Display Conventions and Configuration</H2>

<P>
<!--This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
-->
<P>To show only selected subtracks, uncheck the boxes next to the tracks that
you wish to hide.</P>

<DL>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based on
processed data. Signals displayed in this track are the results of pooled replicates.</DD>
</DL>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<P>
To isolate mononucleosome core DNA fragments from the GM12878 and K562
ENCODE cell lines we followed the micrococcal nuclease (MNase) digestion and
isolation protocol as described in Johnson et al. (2006), Valouev et al. (2008), and
Valouev et al. (2011) with the following modifications. The precise concentrations of the
two flash-frozen cell samples received from the Snyder Lab were not known so, per our
standard procedure, we performed a series of digestions titrating the amount of MNase
to determine the concentration of MNase for optimal digestion of each sample. Final
concentrations of 25 U/&mu;L and 50 U/&mu;L of MNase were used to digest the GM12878
cells and K562 cells respectively at 20&deg;C for 12 min. All other steps in the digestion and
isolation protocol were as described.
</P>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
K562 and GM12878 were each grown to &#126;2.5&times;10<sup>8</sup> cells.  The cells were harvested, frozen and 
the nucleosome core isolation followed (Valouev et al. 2008).
</P>

<P>The SOLiD reads were mapped in color-space with the probabilistic mapper,
<A HREF="https://dnanexus.com" TARGET=_BLANK>DNAnexus</A>. The DNAnexus mapper measures and propagates mapping uncertainty by including both quality values and mismatches in the alignment score calculation. The scores are then scaled across all possible mappings of the read to estimate the posterior probability for alignment to each genomic location. Reads corresponding to posterior probability of correct mapping &gt; 0.9 were reported. 
</P>

<P>Nucleosome density signal maps (bedgraph and bigwig files) were generated by 
first shifting reads by 74 bp in the 5&acute; to 3&acute; direction and counting 
the total number of reads starting at each genomic coordinate on both strands. 
These counts are then smoothed using un-normalized kernel density smoothing 
with a triweight kernel. A bandwidth of 30 bp is used which is equivalent to 
a smoothing window of 60 bp. The smoothed counts at each position are then 
divided by the expected number of reads from an equivalent uniform distribution 
of reads in a &plusmn; 30 bp window around that position. If less than 25&#37; 
of the positions in a &plusmn; 30 bp window around a genomic location are 
uniquely mappable or if the location is part of an assembly gap, the signal 
value at that position is considered unreliable and not recorded in the signal 
files. Hence, genomic coordinates that do not have any associated signal value 
should be considered missing or unreliable data. Genomic coordinates associated 
with a signal value of 0 are reliably mapable but do not have any signal in the dataset.
</P>

<H2>Verification</H2>

<P>The data were validated by pooling together separate runs
to give sufficient sequencing depth for a reasonable signal
to noise ratio. Verification that each replicate
shows similar strand-cross correlation profiles with a strong
peak at approximately 147 bp indicates that the predominant fragment
length is equal to the typical size of a mononucleosome.
</P>

<H2>Credits</H2>

<P><A HREF="mailto:&#109;ps&#110;&#121;&#100;&#101;&#114;&#64;&#115;&#116;&#97;&#110;f&#111;&#114;&#100;.
e&#100;&#117;">
Michael Snyder</A> (PI, Stanford University, California)
<!-- above address is mpsnyder at stanford.edu -->
</P>
<P>
Debasish Raha (Snyder lab) cell culture
</P>

<P><A HREF="mailto:&#115;&#116;ev&#101;&#110;&#106;&#64;&#98;&#121;u.
&#101;d&#117;">
Steven Johnson</A> (PI, Brigham Young University, Utah)
<!-- above address is stevenj at byu.edu -->
</P>
<P>
Elliot Winters (Johnson lab) prepared the mononucleosome core fragments
</P>

<P><A HREF="mailto:&#97;r&#101;&#110;&#100;&#64;&#115;&#116;&#97;n&#102;&#111;&#114;&#100;.
&#101;&#100;u">
Arend Sidow</A> (PI, Stanford University, California)
<!-- above address is arend at stanford.edu -->
</P>
<P>
Ziming Weng (Sidow lab) built the libraries
</P>
<P>
Cheryl Smith (Sidow lab) sequenced the libraries
</P>
<P>
Philippe Lacroute and Philipe Cayting (Stanford Center for Genomics and Personalized Medicine) managed the data submission
</P>

<P>
Contact:
<A HREF="mailto:&#97;&#107;&#117;&#110;&#100;a&#106;e&#64;&#115;&#116;&#97;&#110;&#102;o&#114;&#100;.
&#101;&#100;u">
Anshul Kundaje</A> (Batzoglou and Sidow labs) mapped the data at DNAnexus and generated the signal tracks
<!-- above address is akundaje at stanford.edu -->
</P>

<H2>References</H2>

<p>
Johnson SM, Tan FJ, McCullough HL, Riordan DP, Fire AZ.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17038564" target="_blank">
Flexibility and constraint in the nucleosome core landscape of Caenorhabditis elegans chromatin</a>.
<em>Genome Res</em>. 2006 Dec;16(12):1505-16.
</p>

<p>
Valouev A, Ichikawa J, Tonthat T, Stuart J, Ranade S, Peckham H, Zeng K, Malek JA, Costa G, McKernan K <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18477713" target="_blank">
A high-resolution, nucleosome position map of C. elegans reveals a lack of universal sequence-dictated positioning</a>.
<em>Genome Res</em>. 2008 Jul;18(7):1051-63.
</p>

<p>
Valouev A, Johnson SM, Boyd SD, Smith CL, Fire AZ, Sidow A.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21602827" target="_blank">
Determinants of nucleosome organization in primary human cells</a>.
<em>Nature</em>. 2011 May 22;474(7352):516-20.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column on the track configuration page and
the download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
