<H2>Description</H2>

<p>This track was produced as part of the ENCODE Project.
This track shows DNaseI sensitivity measured genome-wide in different
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>
using the Digital DNaseI methodology (see below) and DNaseI hypersensitive
sites.
DNaseI has long been used to map general chromatin accessibility and
DNaseI hypersensitivity is a universal feature of active
<EM>cis</EM>-regulatory sequences. The use of this method has led
to the discovery of functional regulatory elements that include enhancers,
insulators, promoters, locus control regions and novel elements.
For each experiment (cell type), this track shows DNaseI sensitivity as a
continuous function using sequencing tag density (<I>Raw Signal</I>) 
and discrete loci of DNaseI sensitive zones (<I>HotSpots</I>) and
hypersensitive sites (<I>Peaks</I>).
</p>

<H2>Display Conventions and Configuration</H2>

<p>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
</p>
<p>
For each cell type, this track contains the following views:
<DL>
<DT><I>HotSpots</I></DT>
<DD> DNaseI sensitive zones identified using the HotSpot algorithm.</DD>
<DT><I>Peaks</I></DT>
<DD> DNaseI hypersensitive sites (DHSs) identified as signal peaks within
FDR 1.0% hypersensitive zones.</DD>
<DT><I>Raw Signal</I></DT>
<DD>The density of tags mapping within a 150 bp sliding window
(at a 20 bp step across the genome).</DD>
</DL>
</p>
<p>DNaseI sensitivity is shown as the absolute density of <EM>in vivo</EM>
cleavage sites across the genome mapped using the Digital DNaseI methodology
(see below).  
</p>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<p>
Cells were grown according to the approved
<A TARGET=_BLANK HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Digital DNaseI was performed by DNaseI digestion of intact
nuclei, isolation of DNaseI 'double-hit' fragments as described in
Sabo <EM>et al.</EM> (2006), and direct sequencing of fragment
ends (which correspond to <EM>in vivo</EM> DNaseI cleavage sites)
using the Illumina Geneome Analyzer II platform (36 bp reads).  Uniquely mapping high-quality reads
were mapped to the genome.  DNaseI sensitivity is directly reflected in
raw tag density (<I>Raw Signal</I>), which is shown in the track as density of tags
mapping within a 150 bp sliding window (at a 20 bp step across the genome).
DNaseI sensitive zones (<I>HotSpots</I>) were identified using the
HotSpot algorithm
described in Sabo <EM>et al.</EM> (2004). False discovery rate thresholds of 1.0% (FDR 1.0%) were computed
 for each cell type by applying the HotSpot algorithm to an equivalent number of random
uniquely mapping 36mers.  DNaseI hypersensitive sites (DHSs or <I>Peaks</I>)
were identified as signal peaks within FDR 1.0% hypersensitive
zones using a peak-finding algorithm.</p>

<H2>Verification</H2>

<p>Data were verified by sequencing biological replicates displaying a 
correlation coefficient &gt; 0.9.  Results were extensively validated by
conventional DNaseI hypersensitivity assays using end-labeling/Southern
blotting methods. </p>

<h2>Release Notes</h2>

<P>
This is release 6 (July 2012) of this track. It includes 11 new experiments across 12 new cell lines: bone marrow HS27a, bone marrow HS5, bone marrow MSC, CD4+ Naive Wb11970640, 
CD4+ Naive Wb78495824, Th17, Th1 Wb33676984, Th1 Wb54553204, Th2 Wb33676984, Th2 Wb54553204, Treg Wb78495824, Treg Wb83319432.
This release also removes previously release K562 zinc-finger experiments. There are questions concerning the data due to a merging issue.
</P>

<P>

<H2>Credits</H2>

<p>These data were generated by the UW ENCODE group. </p>
<p>Contact: <A HREF="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">
Richard Sandstrom</A>
<!-- above address is sull at u.washington.edu -->
</p>

<H2>References</H2>

<p>
Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J, Hall R, Mack J, Dorschner MO <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/15550541" target="_blank">
Discovery of functional noncoding elements by digital analysis of chromatin structure</a>.
<em>Proc Natl Acad Sci U S A</em>. 2004 Nov 30;101(48):16837-42.
</p>

<p>
Sabo PJ, Kuehn MS, Thurman R, Johnson BE, Johnson EM, Cao H, Yu M, Rosenzweig E, Goldy J, Haydock A <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16791208" target="_blank">
Genome-scale mapping of DNase I sensitivity in vivo using tiling DNA microarrays</a>.
<em>Nat Methods</em>. 2006 Jul;3(7):511-8.
</p>

<H2>Data Release Policy</H2>

<p>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<a href="../ENCODE/terms.html" target="_blank">here</A>.</p>
