<H2>Description</H2>

<P>This track was produced as part of the ENCODE Project.
This track displays maps of genome-wide binding of the CTCF transcription factor in different 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>
using ChIP-seq high-throughput sequencing.

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
<P>
For each cell type, this track contains the following views:
<DL>
<DT><I>HotSpots</I>
<DD> ChIP-seq affinity zones identified using the HotSpot algorithm.
<DT><I>Peaks</I>
<DD> ChIP-seq affinity sites identified as signal peaks within
affinity zones (FDR 1.0%).
<DT><I>Raw Signal</I>
<DD> The density of tags mapping within a 150 bp sliding window
(at a 20 bp step across the genome).
</DL>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
Cells were crosslinked with 1% formaldehyde, and the reaction was quenched
by the addition of glycine. Fixed cells were rinsed with PBS, lysed in nuclei
lysis buffer, and the chromatin was sheared to 200-500 bp fragments using
Fisher Dismembrator (model 500). Sheared chromatin fragments were
immunoprecipitated with specific polyclonal antibodies at 4 &deg;C
with gentle rotation. Antibody-chromatin complexes were washed and eluted.
The cross linking in immunoprecipitated DNA was reversed and treated with
RNase-A.  Following proteinase K treatment, the DNA fragments were purified
by phenol-chloroform-isoamyl alcohol extraction and ethanol precipitation.
A quantity of 20-50 ng of ChIP DNA was end-repaired, adenine ligated to Illumina adapters was added, and a
Solexa library was made for sequencing.
<P>
ChIP-seq affinity was directly measured through
raw tag density (<I>Raw Signal</I>), which is shown in the track as density of tags
mapping within a 150 bp sliding window (at a 20 bp step across the genome).
ChIP-seq affinity zones (<I>HotSpots</I>) were identified using the
HotSpot algorithm described in Sabo <EM>et al.</EM> (2004);
1.0% false discovery rate thresholds (FDR 0.01) were computed for each cell type by
applying the HotSpot algorithm to an equivalent number of random
uniquely mapping 36mers. ChIP-seq affinities (<I>Peaks</I>)
were identified as signal peaks within affinity
zones (FDR 1.0%) using a peak-finding algorithm.
</P>
<P>
All tracks have a False Discovery Rate of 1% (FDR 1.0%).
</P>

<H2>Verification</H2>

<P>
Data were verified by sequencing biological replicates displaying correlation
coefficients &gt; 0.9.
</P>

<H2>Release Notes</H2>

<P>
Release 3 (February 2012) contains 3 new experiments. 
</P> 

<H2>Credits</H2>

<P>These data were generated by the UW ENCODE group. </P>
<P>Contact: <A HREF="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">Richard Sandstrom</A>
<!-- above address is sull at u.washington.edu -->
</P>

<H2>References</H2>

<p>
Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J, Hall R, Mack J, Dorschner MO <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/15550541" target="_blank">
Discovery of functional noncoding elements by digital analysis of chromatin structure</a>.
<em>Proc Natl Acad Sci U S A</em>. 2004 Nov 30;101(48):16837-42.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<a href="../ENCODE/terms.html" target="_blank">here</a>.</P>
