<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML>
<body>

<h2>Description</h2>

<p>The ENCODE project has revealed the functional elements of segments
of the human genome in unprecedented detail. However, the ability to
distinguish between transcripts designated for translation into protein
and those that serve purely regulatory roles remains elusive. A
standard means to determine if translation is occuring is to measure 
protein produced by transcripts via mass spectrometry-based proteogenomic
mapping. In this process, proteins were digested to peptides using a protease 
such as trypsin and these petides were chromatographically fractionated and
fed into a tandem mass spectrometer (MS/MS). This process creates a signature 
series of fragment masses which can be scanned against the theoretical 
translation and proteolytic digest of an entire genome to identify the genomic 
origins of sample proteins (Giddings <EM>et al.</EM>, 2003).</p>

<p>This proteogenomic track displays mass spectrometry data that have been 
matched to genomic sequences for selected cell lines, using a workflow and 
software specifically designed for this purpose.  The track can be used to 
identify which parts of the genome are translated into proteins, to verify 
which transcripts discovered by other ENCODE experiments are protein-coding,
to reveal new genes and/or splice variants and proteins with post-translational 
modifications (PTM). Of particular interest is the possibility
of uncovering the translation of small open reading frames (ORFs), antisense 
transcripts, or protein-coding regions that have been annotated as introns 
previously.
</p>

<h2>Display Conventions and Configuration</h2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
To show only selected subtracks, uncheck the boxes next to the tracks that
you wish to hide. Color differences among the views are arbitrary. They provide a
visual cue for distinguishing between the different cell types and compartments.
Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<p>This track shows peptide mappings as contiguous rectangular
items rendered in grayscale according to their
score, with darker items representing higher-confidence peptide mappings.
The name of each item is the amino acid sequence of the peptide where a period (.)
at the end of a name signifies a stop codon.
</p>

<dl>
<dt><i>Peptide Genome and GENCODE Mapping(Filtered)</i>
</dt><dd> Peptide mapping results based on hg19 and GENCODE annotation for mass-spectrometry-based proteomics experiments filtered for a false discovery rate (FDR) better than 5%. Specific field descriptions can be found below.
</dd><dt><i>Modified Peptide Genome and GENCODE Mapping(Filtered)</i>
</dt><dd>Modified peptide mapping results based on hg19 and GENCODE annotation for mass-spectrometry-based proteomics experiments filtered for a false discovery rate (FDR) better than 5%.
</dd>
</dl>

<P>
<!--echo var="BROWSER_HOST" -->
Unfiltered <A TARGET=_BLANK HREF="../cgi-bin/hgEncodeVocab?tag=MPEPMAPGCUNFT,PEPMAPGCUNFT">views</A> are available on the Downloads page.
</P>








<p>Fields specific to Proteogenomic tracks include:<br></p>
<ul>
  <li>The <span style="font-style: italic;">Item</span> names the peptide sequence and
is appended with a number for proteins with post-translational modifications (PTM)
representing the integer portion of the PTM mass. The peptide sequence appears as a short label beside
the main Genome Browser display window depending on the view configuration.</li>
  <li>The <span style="font-style: italic;">Score</span> is used to render shade to
displayed rectangular items and is derived from the <span 
style="font-style: italic;">rawScore </span>(see below) given by the proteomics 
peptide mapping software Peppy.  It is computed as [(<span style="font-style: italic;">rawScore</span>
- <span style="font-style: italic;">rawScore</span> at 10% FDR cutoff) /
(<span style="font-style: italic;">rawScore</span> at near 0% FDR cutoff -
<span style="font-style: italic;">rawScore</span> at 10% FDR cutoff)] * 1000,
and is then converted to an integer.  Raw scores above the 0% FDR threshold have
a score of 1000 (best), while those below the 10% FDR threshold have a score of 0 (worst).</li>
  <li>The <span style="font-style: italic;">rawScore</span> is given by Peppy 
and is expressed as the negative log 10 of the <EM>p-value</EM>, which reflects
the confidence of the mapping between the peptides and the spectrums.
On the item details pages, <EM>rawScore</EM> is labeled: Raw score for a peptide/spectrum match.</li>
  <li>The <span style="font-style: italic;">spectrumId</span> is an identifier of the
spectrum associated with the peptide mapping and can be used to track the original spectrum.
On the item details pages, <EM>spectrumId</EM> is labeled: An identifier of the spectrum
associated with the peptide mapping.</li>
  <li>The <span style="font-style: italic;">peptideRank</span> is a rank of the
peptide/spectrum match used for a spectrum matching to different peptides. A spectrum
can be chimeric (containing more than one peptide) and the spectrum can be mapped
to two or more distinct peptides. Here, only the top-scoring match is reported. If more
than one peptide "tied" for the top score, then all peptides were included and all 
matches have a <span style="font-style: italic;">peptideRank</span> of 1. On the
item details pages, <EM>peptideRank</EM> is labeled: Rank of the peptide/spectrum match,
for spectrum matching to different peptides.</li>
  <li>The <span style="font-style: italic;">peptideRepeatCount</span> indicates the
number of places in the genome where the peptide is encoded for a peptide/spectrum
match. It reflects the prevalence or uniqueness of the peptide mapping in the genome.
Those peptides mapped to only a few genomic locations will have a low
<span style="font-style: italic;">peptideRepeatCount</span>, whereas those peptides mapped
to highly duplicated regions will have a high <span style="font-style:
italic;">peptideRepeatCount</span>. <b>Peptides with
a <EM>peptideRepeatCount</EM> greater than 10 times in the genome were
deleted from the track</b> (this field is for regular peptides only). On the 
item details pages, <EM>peptideRepeatCount</EM> is labeled: Indicates the number
of places in the genome where the peptide is encoded for a peptide/spectrum match.</li>
  <li>The <span style="font-style: italic;"> modificationMass</span> reflects the 
additional molecular weight for each modified peptide matched to a spectrum (this
field is for PTM peptides only). On the item details pages, <EM>modificationMass</EM> is labeled:
Reflects the additional molecular weight for each modified peptide matched to a spectrum.</li>
</ul>


<h2>Methods</h2>

<p>ENCODE cell lines K562, GM12878, H1-hESC and H1-neurons were used for this 
large scale proteomic analysis. Cell lines were cultured according to standard
<A HREF="/ENCODE/protocols/cell" target="_blank">ENCODE cell culture protocols</A>
and tryptic peptides were prepared using In-gel digestion (Shevchenko <EM>et al.,
</EM> 2007), FASP (Wiseniewski <EM>et al.</EM>, 2009; Manza <EM>et al.,</EM> 2005) 
or MudPIT (Washburn <EM>et al.</EM>, 2001) protocols as indicated for each sample.  
Tandem mass spectrometry (RPLC-MS/MS) analysis was then performed on an Eksigent 
Ultra-LTQ Orbitrap system or a Q Exactive system (Thermo Scientific) as indicated.* 
The number of arginine or lysine sites missed by the trypsin enzyme is indicated by 
the metadata parameter <i>miscleavages</i>.

<p>We performed proteogenomic mapping (Jaffe<EM> et al.,</EM> 2004) on an <i>in silico</i> 
translation and proteolytic digestion of the whole human genome (UCSC Hg19), and 
the GENCODE translation of protein-coding transcripts database with up to 
one missed cleavage using <A HREF="http://www.geneffects.com/peppy" target="_blank">
Peppy software</A>. The GENCODE version for H1-hESC (FASP protocol), K562, and 
GM12878 is V11 and it is V10 for H1-hESC (MudPIT protocol) and H1-neurons datasets.  
GENCODE V11 was initially used for database search and it was later found that 
GENCODE V10 is the preferred version and was subsequently used to replace GENCODE 
V11 for the analyses of the later datasets.  Peppy's embedded algorithm matches 
the MS/MS spectra to peptides and outputs a matching score, and the peptides are 
then mapped back to their corresponding genomic sequences.  The peptide/spectrum 
matches (PSMs) found from Hg19 genome and GENCODE searches were compared and the PSMs 
of higher score from either matches were reported.  If the scores from both matches 
are equal, both of them were reported.  Additional peptides matches were found by 
GENCODE search that were not found in Hg19 genome search, some of which span slice 
junctions.  Overall, a cross-comparison and inclusion of results from both database 
searches resulted in a greater coverage.

<p>
For both the Hg19 genome and GENCODE database searches, a blind search for 
post-translational modifications (PTMs) was performed using Peppy software.  
In a blind PTM search, when Peppy matches a MS/MS spectrum to a peptide, if the 
matching score is increased after the addition of the molecular weight (MW) of a 
potential PTM, the peptide is determined as having a PTM.  In the output of both the
Hg19 genome and GENCODE searches, some spectra were output as matched with peptides 
of PTMs and others were output as matched with regular peptides, <EM>i.e.</EM>, peptides 
without PTMs.  Once the best-ranking PSMs were identified from either search, the
regular peptides and peptides with PTMs were displayed in separate tracks. 
</p>

<p>
For each data set, a reverse database search was also performed using all spectra
to calculate the false discovery rate (FDR)  (Elias <EM>et al.,</EM> 2007).  Only
those matches with a FDR rate below 5% were included in this track.  The
unfiltered results of those peptides matches with an FDR rate below 10% are available 
for download.
</p>

*<font size="-1">H1-hESC (FASP protocol), K562 and GM12878 samples were analyzed on the Eksigent 
Ultra LTQ Orbitrap system (Thermo Scientific) whereas H1-hESC (MudPIT protocol), 
H1-neurons sample were analyzed on the Q Exactive system (Thermo Scientific).</font>

<h2>Release Notes</h2>
This is Release 1 of this track (Sept 2012).  Unlike other ENCODE data, these data are not archived at GEO but at <a href="https://proteomecommons.org/" target="_blank">Proteome Commons</a>.  The first 32 digits of the Tranche Hash for each data set is stored as the labExpId.

<h2>Credits</h2>
<p>Proteogenomic mapping: Dr. John Wrobel, Dr. Jainab Khatun, Mr. Brian Risk, 
and Mr. David Thomas (<a href="http://giddingslab.org/" target="_blank">Giddings Lab</a>). </p>
<p>Proteomic analysis: Dr. Yanbao Yu, Dr. Harsha Gunawardena, Dr. Ling Xie and 
Ms. Li Wang (<a href="http://www.med.unc.edu/biochem/chenlab/chen-research-lab" target="_blank">Chen Lab</a>).
</p>
Main Contact:
<a href="mailto:johnwrobel@boisestate.edu">John Wrobel</a>
<!-- above address is khatun at email.unc.edu -->

<h2>References</h2>

<p>
Giddings MC, Shah AA, Gesteland R, Moore B.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/12518051" target="_blank">
Genome-based peptide fingerprint scanning</a>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Jan 7;100(1):20-5.
</p>

Shevchenko A, Tomas H, Havlis J, Olsen JV, Mann M.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17406544" target="_blank">
In-gel digestion for mass spectrometric characterization of proteins and proteomes</a>.
<em>Nat Protoc</em>. 2006;1(6):2856-60.
</p>

<p>
Wisniewski JR, Zougman A, Nagaraj N, Mann M.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19377485" target="_blank">
Universal sample preparation method for proteome analysis</a>.
<em>Nat Methods</em>. 2009 May;6(5):359-62.
</p>

<p>
Manza LL, Stamer SL, Ham AJ, Codreanu SG, Liebler DC.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/15761957" target="_blank">
Sample preparation and digestion for proteomic analyses using spin filters</a>.
<em>Proteomics</em>. 2005 May;5(7):1742-5.
</p>

<p>
Washburn MP, Wolters D, Yates JR 3rd.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/11231557" target="_blank">
Large-scale analysis of the yeast proteome by multidimensional protein identification
technology</a>.
<em>Nat Biotechnol</em>. 2001 Mar;19(3):242-7.
</p>

<p>
Jaffe JD, Berg HC, Church GM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/14730672" target="_blank">
Proteogenomic mapping as a complementary method to perform genome annotation</a>.
<em>Proteomics</em>. 2004 Jan;4(1):59-77.
</p>

<p>
Elias JE, Gygi SP.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17327847" target="_blank">
Target-decoy search strategy for increased confidence in large-scale protein identifications by mass
spectrometry</a>.
<em>Nat Methods</em>. 2007 Mar;4(3):207-14.
</p>

 
<h2>Data Release Policy</h2>

<p>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset
until
nine months following the release of the dataset. This date is listed
in
the <em>Restricted Until</em> column above. The full data release policy for ENCODE is available
<a href="../ENCODE/terms.html" target="_BLANK">here</a>.</p>
</body>
</html>

<BR>

	</TD><TD WIDTH=15></TD></TR></TABLE>
	</TD></TR></TABLE>
	</TD></TR></TABLE>
	
</TD></TR></TABLE>

</BODY></HTML>
