<H2>Description</H2>

<P>This track, produced as part of the ENCODE Project, contains deep sequencing DNase data
that will be used to identify sites where regulatory factors bind to the genome
(<i>footprints</i>).</P>
<P>Footprinting is a technique used to define the DNA sequences that interact
with and bind DNA-binding proteins, such as transcription factors,
zinc-finger proteins, hormone-receptor complexes, and other 
chromatin-modulating factors like CTCF.  The technique depends upon the
strength and tight nature of protein-DNA interactions. In their native chromatin
state, DNA sequences that interact directly with DNA-binding proteins are
relatively protected from DNA-degrading endonucleases, while the exposed/unbound
portions are readily degraded by such endonucleases. A massively parallel
next-generation sequencing technique to define the DNase hypersensitive sites
in the genome was adopted. The DNase samples were sequenced using next-generation
sequencing machines to significantly higher depths of 300-fold or greater. This produces
a base-pair level resolution of the DNase susceptibility maps of the native
chromatin state.  These base-pair resolution maps represent and are dependent
upon the nature and the specificity of interaction of the DNA with the
regulatory/modulatory proteins binding at specific loci in the genome; thus
they represent the native chromatin state of the genome under investigation.
The deep sequencing approach has been used to define the footprint landscape of
the genome by identifying DNA motifs that interact with known or novel DNA-binding proteins.
<P>

<H2>Display Conventions and Configuration</H2>

<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>). For each view, there are multiple subtracks that display
individually on the browser. Instructions for configuring multi-view tracks
are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
<P>
For each cell type, this track contains the following views:
<DL>
<DT><I>HotSpots</I>
<DD> DNaseI hypersensitive zones identified using the HotSpot algorithm.
<DT><I>Peaks</I>
<DD> DNaseI hypersensitive sites (DHSs) identified as signal peaks within 
FDR 1.0% hypersensitive zones.
<!-- footprint track is expected in the future
<DT><I>Footprint</I>
<DD>While the HotSpots algorithm identifies
genomic regions of elevated tag counts relative to random expectation,
the <i>FP-DETECTOR</i> algorithm (developed in our center) searches for
smaller locations where proteins are most likely bound within the
identified HotSpot regions.  
</DT>
-->
<DT><I>Signal</I>
<DD>Per-base count of sequence reads whose 5' end (corresponding to a 
DNaseI-induced DNA cut) coincides with the given position.
</DT>
<DT><I>Raw Signal</I>
<DD>The density of tags mapping within a 150 bp sliding window
(at a 20 bp step across the genome).
</DL>
NOTE: The names of the signal views in this track are reversed from conventions used in
other ENCODE tracks, where the less processed signal is termed 'Raw'.
<P>
<P>DNaseI sensitivity is shown as the absolute density of <EM>in vivo</EM> 
cleavage sites across the genome mapped using the Digital DNaseI methodology 
(see below).  
</P>

<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>
<!-- TODO
DNaseI hypersensitive sites (DHSs) at False Discovery Rate (FDR) 0.5% are shown 
within each density track as colored red subregions.</P>
-->

<H2>Methods</H2> 

Cells were grown according to the approved <A TARGET=_BLANK
HREF="/ENCODE/protocols/cell/" TARGET=_BLANK>ENCODE cell culture
protocols</A>.  Digital DNaseI was performed by DNaseI digestion of
intact nuclei, followed by isolation of DNaseI 'double-hit' fragments as
described in Sabo <EM>et al.</EM> (2006), and direct sequencing of
fragment ends (which correspond to <EM>in vivo</EM> DNaseI cleavage
sites) using the Solexa platform (27 bp reads).  High-quality reads were
mapped to the GRCh37/hg19 human genome using <a href="http://bowtie-bio.sourceforge.net/index.shtml" target="_blank">Bowtie 0.12.5</a> 
(Eland was used to map to NCBI36/hg18);
only unique mappings were kept. DNaseI sensitivity is directly reflected 
in raw tag density (<I>Signal</I>), which is shown
in the track as density of tags mapping within a 150 bp sliding window
(at a 20 bp step across the genome).  DNaseI hypersensitive zones
(<I>HotSpots</I>) were identified using the HotSpot algorithm
described in Sabo <EM>et al.</EM> (2004).  False discovery rate thresholds 
of 1.0% (FDR 1.0%) were computed for each cell type by applying
the HotSpot algorithm to an equivalent number of random uniquely
mapping 36-mers.  DNaseI hypersensitive sites (DHSs or <I>Peaks</I>)
were identified as signal peaks within 1.0% (FDR 1.0%) hypersensitive zones
using a peak-finding algorithm.  Only DNase Solexa libraries from unique
cell types producing the highest quality data, as defined by Percent
Tags in Hotspots (PTIH ~40%), were designated for deep sequencing to a depth
of over 200 million tags. 
<!-- footprint track is expected in the future
While the HotSpots algorithm identifies
genomic regions of elevated tag counts relative to random expectation,
the <i>FP-DETECTOR</i> algorithm (developed by the UW ENCODE group) searches for
smaller locations where proteins are most likely bound within the
identified HotSpot regions.  The reported regions marked as <i>footprints</i>
are disjoint, 6 to 40 base-pairs in length, and show a marked
depletion of tag counts relative to their respective, local
backgrounds.  
-->

</P>

<H2>Verification</H2>

<P>
Results were validated by 
conventional DNaseI hypersensitivity assays using end-labeling/Southern 
blotting methods. 
</P>
<P> Images and their associated mappings can be found in the <A HREF="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeUwDgf/supplemental/">supplemental data</A>. 

<H2>Release Notes</H2>

<P>
This is Release 4 (August 2012) of this track, which includes 10 new experiments across 8 cell lines.
</P>
<P>
A number of previously released Peaks have been replaced by updated versions.
The affected database tables and files include 'V2' in the name, and metadata is marked with "submittedDataVersion=V2",
followed by the reason for replacement, "Fixed bug in peak calls that artificially reduced the number of peaks".
</P>
<P>
Previous versions of files are available for download from the
<A HREF="ftp://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeUwDgf/"
 TARGET=_BLANK>FTP site</A>.
 </P>

<H2>Credits</H2>

<P>These data were generated by the UW ENCODE group. </P>
<P>Contact: <A HREF="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">
Richard Sandstrom</A>
<!-- above address is sull at u.washington.edu -->
</P>

<H2>References</H2>

<p>
Sabo PJ, Hawrylycz M, Wallace JC, Humbert R, Yu M, Shafer A, Kawamoto J, Hall R, Mack J, Dorschner MO <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/15550541" target="_blank">
Discovery of functional noncoding elements by digital analysis of chromatin structure</a>.
<em>Proc Natl Acad Sci U S A</em>. 2004 Nov 30;101(48):16837-42.
</p>

<p>
Sabo PJ, Kuehn MS, Thurman R, Johnson BE, Johnson EM, Cao H, Yu M, Rosenzweig E, Goldy J, Haydock A <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16791208" target="_blank">
Genome-scale mapping of DNase I sensitivity in vivo using tiling DNA microarrays</a>.
<em>Nat Methods</em>. 2006 Jul;3(7):511-8.
</p>

<H2>Data Release Policy</H2>

Data users may freely use ENCODE data, but may not, without prior consent, submit publications that use an unpublished ENCODE dataset until nine months following the release of the dataset. This date is listed in the Restricted Until column, above.
The full data release policy for ENCODE is available <a href="/ENCODE/terms.html" target="_blank">here</a>.
