<H2>Description</H2>

<P>
This track was produced as part of the ENCODE Project. 
It shows the locations of protein factor
mediated chromatin interactions determined by 
Chromatin Interaction Analysis with Paired-End Tag (ChIA-PET) data
(Fullwood et al., 2010)
extracted from five different human cancer cell lines 
(K562 (chronic myeloid leukemia), HCT116 (colorectal cancer),
 HeLa-S3 (cervical cancer), MCF-7 (breast cancer), and NB4
(promyelocytic)). A chromatin interaction is
defined as the association of two regions of the genome that are 
far apart in terms of genomic distance, but are
spatially proximate to each other in 
the 3-dimensional cellular nucleus.
<P>
Additionally, ChIA-PET experiments generate 
transcription factor binding sites. A binding site
is defined as a region of the genome 
that is highly enriched by specific Chromatin
ImmunoPrecipitation (ChIP) against a transcription 
factor, which indicates that the transcription
factor binds specifically to this region. 
The protein factors displayed in the track
include estrogen receptor alpha (ER&#945;), RNA polymerase II (RNAPII),
and CCCTC binding factor (CTCF).
<!- Generally, a chromatin interaction is more likely to
connect two transcription factor binding sites together.
->
<h2>Display Conventions and Configuration</h2>

<P>
In the graphical display, the PETs are 
represented by two blocks one for each end.  
These blocks are  connected by a horizontal
line if both ends are in the same chromosome. 
If the two ends are on different chromosomes, only
one block will display.
PET sequences that overlap at both ends form 
PET clusters. The number of PETs in a cluster reflects
the strength of a chromatin interaction. 
Singleton PETs (PETs without a cluster) are
potentially false positives, whereas PET clusters of 
more than 3 PETs could indicate genuine
chromatin interactions. The density graph of the 
tags shows the ChIP enrichment at different points
of genome, and high peaks indicate transcription
factor binding sites.
</P>
Instructions for configuring multi-view tracks are 
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.

<p>To show only selected subtracks, uncheck the boxes next to the tracks that
you wish to hide.</p>

<dl>
<dt><i>Interactions</i>
</dt><dd>ChIA-PET Chromatin Interaction PET clusters: 
Two different genomic regions in the chromatin are 
genomically far from each other or in different 
chromosomes, but are spatially close to each 
other in the nucleus and interact with each 
other for regulatory functions. 
BED12 format is used to represent the data. 
</dd><dt><I>Signal</I>
<dd>Density graph (wiggle) of signal enrichment based on aligned read density.
</dd></dl>
<p>Metadata for a particular subtrack can be found 
by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<P>
Chromatin interaction analysis with paired-end tag 
sequencing (ChIA-PET) is a global de novo
 high-throughput method for characterizing the 3-dimensional
structure of chromatin in the nucleus. In the 
ChIA-PET protocol, samples were cross-linked and
fragmented, then subjected to chromatin immunoprecipitation. 
The DNA fragments that were brought together 
by the chromatin interactions were then 
proximity-ligated. During this proximity-ligation 
step, the half-linkers (created by the 
fragmentation) containing flanking 
MmeI sites (type IIS restriction enzymes) were
first ligated to the DNA fragments and then
ligated to each other to form full
linkers. Full linkers bridge either two ends of a 
self-circularized fragment, or two ends of two
different chromatin fragments. The material was 
then reverse cross-linked, purified and digested
with MmeI. MmeI cuts 20 base pairs away from its 
recognition site. Tag-linker-tag (paired-end
 tag, PET) constructs were sequenced by ultra-high-throughput
methods (Illumina or SOLiD paired-end sequencing). 
ChIA-PET reads were processed with
 the ChIA-PET Tool (Li et al., 2010)
by the following steps: linker filtering, 
short reads mapping, PET classification, binding site 
identification, and interaction cluster identification. 
The high-confidence binding sites and 
chromatin interaction clusters were reported.

<H2>Verification</H2>

<P>
Chromatin interactions identified by ChIA-PET have been 
validated by 3C, ChIP-3C, 4C and DNA-FISH (Fullwood et al., 2009).

<H2>Credits</H2>

<P>
Genome Institute of Singapore: Guoliang Li, Xiaoan Ruan, Kuljeet Singh Sandhu, Fabianus Hendriyan Mulawadi, Huay Mei Poh, Yufen Goh, Su Qin Peh, Wing-Kin Sung, Yijun Ruan
</P>
<P>
Stanford University: Raymond Auerbach, Michael Snyder</P>

Contact:
<A HREF="mailto:&#114;&#117;a&#110;&#121;j&#64;&#103;&#105;&#115;.
&#97;&#45;&#115;&#116;&#97;&#114;.
&#101;&#100;&#117;.
&#115;g">
Yijun Ruan</A>
<!-- above address is ruanyj at gis.a-star.edu.sg -->

<H2>References</H2>

<p>
Fullwood MJ, Han Y, Wei CL, Ruan X, Ruan Y.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20069536" target="_blank">
Chromatin interaction analysis using paired-end tag sequencing</a>.
<em>Curr Protoc Mol Biol</em>. 2010 Jan;Chapter 21:Unit 21.15.1-25.
</p>
<p>
Fullwood MJ, Liu MH, Pan YF, Liu J, Xu H, Mohamed YB, Orlov YL, Velkov S, Ho A, Mei PH <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19890323" target="_blank">
An oestrogen-receptor-alpha-bound human chromatin interactome</a>.
<em>Nature</em>. 2009 Nov 5;462(7269):58-64.
</p>
<p>
Li G, Fullwood MJ, Xu H, Mulawadi FH, Velkov S, Vega V, Ariyaratne PN, Mohamed YB, Ooi HS, Tennakoon C <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20181287" target="_blank">
ChIA-PET tool for comprehensive chromatin interaction analysis with paired-end tag sequencing</a>.
<em>Genome Biol</em>. 2010;11(2):R22.
</p>

<h2>Publications</h2>

<p>
Li G, Ruan X, Auerbach RK, Sandhu KS, Zheng M, Wang P, Poh HM, Goh Y, Lim J, Zhang J <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/22265404" target="_blank">
Extensive promoter-centered chromatin interactions provide a topological basis for transcription regulation</a>.
<em>Cell</em>. 2012 Jan 20;148(1-2):84-98.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
