<H2>Description</H2>
<P>This track is produced as part of the ENCODE Project.
This track displays transcriptional fragments associated with RNA binding proteins in different
<A TARGET="_BLANK" HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>,
using RIP-Chip (Ribonomic) profiling on Affymetrix GeneChip ENCODE 2.0R Tiling Arrays.
The <i>RBP Assoc RNA</i> view shows the genomic location of transcripts
associated with the array probes.
Data for this track was produced as part of the
<A TARGET="_BLANK" HREF="/ENCODE/index.html">
Encyclopedia of DNA Elements (ENCODE) Project</A>.

</P>


<P>In eukaryotic organisms, gene regulatory networks require an additional
level of coordination that links transcriptional and post-transcriptional
processes. Messenger RNAs have traditionally been viewed as passive
molecules in the pathway from transcription to translation. However,
it is now clear that RNA-binding proteins play a major role in
regulating multiple mRNAs in order to facilitate gene expression patterns.
These tracks show the associated mRNAs that co-precipitate with the
targeted RNA-binding proteins using RIP-Chip profiling.
</P>


<H2>Display Conventions and Configuration</H2>
<P>This track is a multi-view composite track. For each view there
are multiple subtracks that display individually in the browser.
The subtracks within this track correspond to different antibodies/target
proteins tested in different cell lines.
This track is initially released with a single view:
</P>
<DL>
<DT><i>RBP Assoc RNA</i>
<DD>The RBP Assoc RNA view shows the genomic extent of the transcriptional segments
associated with the Affymetrix Tiling Array probes.
</DL>
Instructions for configuring multi-view tracks are

<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.

<H2>Methods</H2>
<P>RBP-mRNA complexes were purified from cells grown according to the approved
<A HREF="/ENCODE/protocols/cell">ENCODE cell culture protocols </A>.
The associated transcriptional fragments were identified using Affymetrix GeneChip ENCODE 2.0R Tiling Arrays.
Arrays were analyzed using Affymetrix Tiling Analysis Software (TAS) version 1.1.
</P>
<P>
Total Input and T7Tag (negative control)
tracks were each developed by single sample analysis of their respective triplicate sets with quantile normalization and linear
scaling applied.  Probe signals were calculated using the Hodges-Lehmann estimator with a bandwidth of 100 bases.
Resulting signal files were examined to develop percentiles.  Interval analysis was performed using a minimum cut-off
equal to the 95th percentile signal score, a minimum run of 21 (3 * platform resolution) and max gap of 63 (9 * platform resolution).
</P>
<P>
Treatment RIP tracks (e.g., ELAVL1, PABC1) were developed by two sample analysis with T7Tag used as
the control.  Quantile normalization and scaling was applied to these sample groups individually.
Probe signals were calculated with the Hodges-Lehmann estimator using signal log (log2) ratios and a bandwidth of 100 bases.
Resulting signal files were examined to develop percentiles.  Interval analysis was performed using a minimum cut-off
equal to the 95th percentile signal ratio score, a minimum run of 21 (3 * platform resolution) and max gap of 63 (9 * platform resolution).

</P>
<P>For additional RIP methods detail, see
Tenenbaum <EM>et al.</EM> 2002; Baroni <EM>et al.</EM> 2008; Penalva
<EM>et al.</EM> 2004, below.



<H2>Verification</H2>

<P>
All experiments (including controls) performed in and analyzed as triplicates.
</P>

<H2>Credits</H2>

<P>These data were produced and analyzed by a collaboration between the
<A HREF="http://www.albany.edu/cancergenomics/faculty/stenenbaum/index.shtml"
TARGET="_BLANK">
Tenenbaum lab</A>
at the University at Albany-SUNY, College of Nanoscale
Science and Engineering, the
<A HREF="http://www.uthscsa.edu/csb/faculty/penalva.asp" TARGET="_BLANK">
Luiz Penalva group</A>
at the Greehey Children's Cancer Research Institute,
University of Texas Health Science Center and the
<A HREF="http://www.cfgbiotech.com/microarray/index.htm" TARGET="_BLANK">
Microarray Core Facility</A> at the Center for Functional Genomics, Rensselaer, NY .
</P>

<P>
Contact:
<A HREF="mailto:&#83;&#84;&#101;&#110;&#101;&#110;&#98;&#97;um&#64;&#117;&#97;m&#97;&#105;&#108;.
&#97;&#108;&#98;&#97;ny.
&#101;&#100;&#117;">
&#83;&#84;&#101;&#110;&#101;&#110;&#98;&#97;um&#64;&#117;&#97;m&#97;&#105;&#108;.
&#97;&#108;&#98;&#97;ny.
&#101;&#100;&#117;</A>
<!-- above address is STenenbaum at uamail.albany.edu -->
<br />


<H2>References</H2>

<P>Tenenbaum SA, Lager PJ, Carson CC, Keene JD.
<A TARGET="_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/12054896">
Ribonomics: identifying mRNA subsets in mRNP complexes using
antibodies to RNA-binding proteins and genomic arrays.</A>
<EM> Methods</EM>. 2002 Feb;26(2):191-8.

</P>

<P>Baroni TE, Chittur SV, George AD, Tenenbaum SA.
<A TARGET="_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/18369977">
Advances in RIP-chip analysis : RNA-binding protein
immunoprecipitation-microarray profiling.</A>
<EM> Methods Mol Biol</EM>. 2008;419:93-108.
</P>

<P>Penalva LO, Tenenbaum SA, Keene JD.
<A TARGET="_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/14770002">
Gene expression analysis of messenger RNP complexes.</A>
<EM> Methods Mol Biol</EM>. 2004;257:125-34.
</P>

<P>Keene JD, Tenenbaum SA.
<A TARGET="_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/12086614">
Eukaryotic mRNPs may represent posttranscriptional operons.</A>
<EM> Mol Cell</EM>. 2002;9(6):1161-7.
</P>

<P>George AD, Tenenbaum SA.
<A TARGET="_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/17114943">
MicroRNA modulation of RNA-binding protein regulatory elements.</A>
<EM> RNA Biol</EM>. 2006;3(2):57-9. Epub 2006 Apr 1.
</P>



<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET="_BLANK" HREF="../ENCODE/terms.html">here</A>.</P>


