<H2>Description</H2>

<P>
The tracks show enrichment of RNA sequence tags generated by
high throughput sequencing (RNA-seq) and mapped to the human
genome. Double stranded cDNA was synthesized from
polyadenylated RNA (polyA+) <!--or, in order to analyze transcription
from RNA polymerase III, from RNA remaining after depletion of
both ribosomal RNA and polyadenylated RNA (polyA-/rRNA-)-->. 
PCR amplified, adapter ligated cDNA, 150-300nt
long, was sequenced on an Illumina GA sequencer.
</p>
<P>
Where designated, cell lines received specific treatments prior to
RNA isolation. As indicated, K562 cells were treated with either
interferon-a or interferon-g for 30 minutes or 6 hours. These
experiments were carried out in conjunction with ChIP-Seq
experiments on the transcription factors STAT1 and STAT2 in order
to examine the effects that inducers of a specific transcriptional
response might have on gene expression and on transcription factor
binding site discovery. K562 cells were treated with a-amanitin in
order to examine the effects of RNA polymerase II inhibition on RNA
polymerase III-mediated transcription.
</p>

<P>This track shows <em>expression</em> data generated as confirmation
of the
<!--<A HREF="hgTrackUi?db=hg19&g=wgEncodeSydhTfbs">SYDH TFBS</A> tracks.-->
<a href="http://genome-preview.ucsc.edu/cgi-bin/hgTrackUi?db=hg19&g=wgEncodeSydhTfbs">SYDH TFBS</a>
tracks currently available on genome-preview.
</P>

	<H2>Display Conventions and Configuration</H2>
<P>
This is a composite track that contains multiple data types
(<EM>views</EM>).  Instructions for configuring composite
tracks are <A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.

<DL>
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based 
on processed data.</DD>
<DT><i>Alignments</i>
</dt><dd>The Alignments view shows reads mapped to the genome and indicates where
bases may mismatch.  The alignment file follows the standard SAM format of Bowtie output with the following additions: the custom tags X0 X1 XN XM XO XG XT XA XS XF XE are present. These tags are described by the <a href="http://bio-bwa.sourceforge.net/bwa.shtml#4" target=blank>BWA specifications</A>.  See the <a href="http://bowtie-bio.sourceforge.net/manual.shtml#sam-bowtie-output" target=_blank>Bowtie Manual</a> for more information about the SAM Bowtie output (including other tags) and the <a href="http://samtools.sourceforge.net/SAM1.pdf#page=4" target=_blank>SAM Format Specification</a> for more information on the SAM/BAM file format. 
</DD>
</DL>

<H2>Methods</H2>
<P>
Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.

Total RNA was extracted using TRIzol reagents
(15596-018, Life Tech), following the manufacturer's protocol.
For polyA+ samples, polyadenylated RNA was purified using the
MicroPoly(A) Purist kit (AM1919, Life Tech) and fragmented using
RNA Fragmentation Reagent (AM8740, Life Tech). Illumina adapters
were ligated to double stranded cDNA which was synthesized using
reagents from Life Tech (11917-010).
<!--For polyA-/rRNA- samples, ribosomal
RNA was removed from total RNA using the RiboMinus Eukaryote
Kit (K1550-02, Life Tech) and polyadenylated RNA was purified
away from the rRNA depleted fraction using the MicroPoly(A) Purist
kit (AM1919, Life Tech). Double-stranded cDNA synthesis was
performed on the rRNA/polyA depleted RNA using random primers
and the SuperScript double-stranded cDNA synthesis kit (11917-010,
Life Tech). -->
</P>
<P>
PCR amplified adapter ligated cDNA (150-300 bp) was sequenced
using Illumina GA. Sequence reads of 27-33nt long with 0-2
mismatches were mapped to the genome. The signal height
corresponds to the number of overlapping fragments at each
nucleotide position in the genome. Samples originally mapped to the
hg18 version of the human genome were remapped to hg19 using
the BWA aligner, version 0.5.7.
</P>

<!-----From the mapped tags, a signal map of ChIP DNA fragments was constructed where 
the signal height is the number of overlapping fragments at each nucleotide 
position in the genome.
---->

<H2>Credits</H2>
<P>These data were generated and analyzed by the labs of
<A HREF="http://snyderlab.stanford.edu/"
  class="external text" title="http://snyderlab.stanford.edu/"
  target="_blank">Michael Snyder</a>,
<A HREF="http://bioinfo.mbb.yale.edu/"
  class="external text"
  title="http://bioinfo.mbb.yale.edu/"
  target="_blank">Mark Gerstein</a> and
<A HREF="http://info.med.yale.edu/genetics//faculty.php?f=49"
  class="external text" title="http://info.med.yale.edu/genetics//faculty.php?f=49"
  target="_blank">Sherman Weissman</a> at Yale University;
<A HREF="http://www.usc.edu/programs/pibbs/site/faculty/farnham_p.htm"
  class="external text" title="www.usc.edu/programs/pibbs/site/faculty/farnham_p.htm"
  target="_blank">Peggy Farnham</a> at USC; and
<A HREF="http://struhl.med.harvard.edu/"
  class="external text" title="http://struhl.med.harvard.edu/"
  target="_blank">Kevin Struhl</a> at Harvard.

<P>
&nbsp;&nbsp;
Contact: <A HREF="mailto:&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;"
  title="&#100;&#97;&#116;&#97;&#115;u&#98;&#109;&#105;&#115;&#115;i&#111;n&#64;&#103;&#101;r&#115;t&#101;&#105;n&#108;&#97;&#98;.&#111;&#114;&#103;" rel="nofollow"
  TARGET = _BLANK>Gerstein Lab</a>.
<!-- above address is datasubmission at gersteinlab.org -->
</P>

<H2>References</H2>

<p>
Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, Gerstein M, Snyder M.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18451266" target="_blank">
The transcriptional landscape of the yeast genome defined by RNA sequencing</a>.
<em>Science</em>. 2008 Jun 6;320(5881):1344-9.
</p>

<p>
Raha D, Wang Z, Moqtaderi Z, Wu L, Zhong G, Gerstein M, Struhl K, Snyder M.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20139302" target="_blank">
Close association of RNA polymerase II and many transcription factors with Pol III genes</a>.
<em>Proc Natl Acad Sci U S A</em>. 2010 Feb 23;107(8):3639-44.
</p>

<p>
Wu JQ, Habegger L, Noisa P, Szekely A, Qiu C, Hutchison S, Raha D, Egholm M, Lin H, Weissman S <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20194744" target="_blank">
Dynamic transcriptomes during neural differentiation of human embryonic stem cells revealed by short, long, and paired-end sequencing</a>.
<em>Proc Natl Acad Sci U S A</em>. 2010 Mar 16;107(11):5254-9.
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column on the track configuration page and
the download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
