<H2>Description</H2>
<P>The RNA binding protein (RBP) associated mRNA sequencing track (RIP-Seq) is produced as part of the <A TARGET=_BLANK HREF="/ENCODE/index.html">
Encyclopedia of DNA Elements (ENCODE) Project</A>.
This track displays transcriptional fragments associated with RBP in 
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A> K562 and GM12878,
using Ribonomic profiling via Illumina SBS.
</P>

<P>In eukaryotic organisms gene regulatory networks require an additional
level of coordination that links transcriptional and post-transcriptional
processes. Messenger RNAs have traditionally been viewed as passive
molecules in the pathway from transcription to translation. However,
it is now clear that RNA-binding proteins play a major role in
regulating multiple mRNAs in order to facilitate gene expression patterns.
These tracks show the associated mRNAs that co-precipitate with the
targeted RNA-binding proteins using RIP-Seq profiling.
</P>


<H2>Display Conventions and Configuration</H2>
<P>This track is a multi-view composite track. For each view there
are multiple subtracks that display individually in the browser.
The subtracks within this track correspond to different antibodies/target
proteins tested in different cell lines.
</P>

<P>
<DL>
<DT><i>Peaks</i>
<DD>The Peaks view shows the genomic extent of the sequencing read peaks.
<P>
<DT><I>Signal</I>
<DD>Density graph of signal enrichment based on a normalized aligned read density (Read Per Million, RPM).  RPM is reported in the score field and is equal to the number of reads at that position divided by the total number of reads divided by one million.  The Signal view is unflitered and displays dense, continuous data as a graph and the RPM measure assists in visualizing the relative amount of a given transcript across multiple samples.</DD>
<P>
<DT><I>Alignments</I>
<DD>The Alignments view shows reads mapped to the genome.
The alignment file follows the standard SAM format of Bowtie output. 
See the <a href="http://bowtie-bio.sourceforge.net/manual.shtml#sam-bowtie-output" target=_blank>Bowtie Manual</a> for more information about the SAM Bowtie output and the <a href="http://samtools.sourceforge.net/SAM1.pdf#page=4" target=_blank>SAM Format Specification</a> for more information on the SAM/BAM file format. 
</DD>
</DL>

Instructions for configuring multi-view tracks are
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.

<H2>Methods</H2>
<P>RBP-mRNA complexes were purified from cells grown according to the approved
<A HREF="/ENCODE/protocols/cell">ENCODE cell culture protocols </A>.
RNA samples were amplified and converted to cDNA with the <A HREF="http://www.nugeninc.com">Nugen</A> Ovation&#169; RNA-Seq System and prepped for sequencing with the
<A HREF="http://www.illumina.com">Illumina</A> mRNA-Seq protocol.  

Approximately 30 million single end sequencing reads were obtained for each K562 and GM12878.
RIP samples were analyzed for signal that was at or above the 60th percentile and statistically enriched compared to the negative control.
Sequences were analyzed using <A HREF="http://tophat.cbcb.umd.edu/">TopHat</A> (Trapnell <EM>et al.</EM>, 2009) with 
<A HREF="http://bowtie-bio.sourceforge.net/index.shtml">Bowtie</A> (Langmead <EM>et al.</EM>, 2009).
</P>
<P>
Peaks were called from the top 40&#37; of TopHat normalized reads,
with a max gap,  min run of (24:48).
Unions of overlapping peak regions from total RNA replicates (RIP-Input) are presented with p-value 
from a one tailed t-test for average signal from replicates versus 0 
(no cut-off was used for totals). Replicate overlap for positive RIP treatment peaks (ELAVL1 and PABPC1)  
are presented with a p-value from one tailed t-test versus signal for same the region in negative 
control replicates (T7-tag). RIP peaks were from sequences longer than 120 bp and p-value &lt; .05.
For both totals (RIP-input) and RIPs, the peak scores are scaled relative p-values between treatment and control.
</P>


<H2>Credits</H2>
<P>These data were produced and analyzed by a collaboration between the
<A HREF="http://www.albany.edu/cancergenomics/faculty/stenenbaum/index.shtml"
TARGET=_BLANK> Tenenbaum lab</A> at the University at Albany-SUNY, College of Nanoscale
Science and Engineering,the <A HREF="http://gccri.uthscsa.edu/lpenalva.asp" TARGET=_BLANK>
Luiz Penalva group</A> at the Greehey Children's Cancer Research Institute,
University of Texas Health Science Center and the <A HREF="http://www.cfgbiotech.com/microarray/index.htm" TARGET=_BLANK>
Microarray Core Facility</A> at the Center for Functional Genomics, Rensselaer, NY.
</P>

<P>
Contact:
<A HREF="mailto:&#83;&#84;&#101;&#110;&#101;&#110;&#98;&#97;um&#64;&#117;&#97;m&#97;&#105;&#108;.
&#97;&#108;&#98;&#97;ny.
&#101;&#100;&#117;">Scott Tenenbaum</A>
<!-- above address is STenenbaum at uamail.albany.edu -->
<br />
</P>


<H2>References</H2>

<P>Baroni TE, Chittur SV, George AD, Tenenbaum SA.
<A TARGET=_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/18369977">
Advances in RIP-chip analysis : RNA-binding protein
immunoprecipitation-microarray profiling
</A>. 
<EM>Methods Mol Biol</EM>. 2008;419:93-108.
</P>

<P>George AD, Tenenbaum SA.
<A TARGET=_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/17114943">
MicroRNA modulation of RNA-binding protein regulatory elements
</A>. 
<EM>RNA Biol</EM>. 2006;3(2):57-9. Epub 2006 Apr 1.
</P>

<P>Keene JD, Tenenbaum SA.
<A TARGET=_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/12086614">
Eukaryotic mRNPs may represent posttranscriptional operons
</A>. 
<EM>Mol Cell</EM>. 2002;9(6):1161-7.
</P>

<P>Langmead B, Trapnell C, Pop M, Salzberg SL.
<A TARGET=_BLANK HREF="http://genomebiology.com/2009/10/3/R25">
<U>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</U>
</A>. 
<EM>Genome Biology</EM>. 2009 Mar; 10:R25.</P>

<P>Penalva LO, Tenenbaum SA, Keene JD.
<A TARGET=_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/14770002">
Gene expression analysis of messenger RNP complexes
</A>. 
<EM>Methods Mol Biol</EM>. 2004;257:125-34.
</P>

<P>Tenenbaum SA, Lager PJ, Carson CC, Keene JD.
<A TARGET=_BLANK" HREF="http://www.ncbi.nlm.nih.gov/pubmed/12054896">
Ribonomics: identifying mRNA subsets in mRNP complexes using
antibodies to RNA-binding proteins and genomic arrays
</A>. 
<EM>Methods</EM>. 2002 Feb;26(2):191-8.
</P>


<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET=_BLANK HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>

