<H2>Description</H2>

<P>These tracks display chromatin immunoprecipitation (ChIP-seq) evidence as
part of the four <B><em>Open Chromatin</em></B> track sets (see below).
ChIP-seq is a method to identify the specific location of proteins that are
directly or indirectly bound to genomic DNA. By identifying the binding location
of sequence-specific transcription factors, general transcription
machinery components, and chromatin factors, ChIP can help in the functional
annotation of the open chromatin regions identified by DNaseI HS mapping and
FAIRE.
</P>
<P>Together with
DNaseI HS and FAIRE experiments, these tracks display the locations of  active
regulatory elements identified as open chromatin in
<A TARGET=ucscVocab HREF="/cgi-bin/hgEncodeVocab?type=cellType">multiple cell types</A>
from the Duke, UNC-Chapel Hill, UT-Austin, and EBI ENCODE group.
Within this project, open chromatin was identified using two
independent and complementary methods: DNaseI hypersensitivity (HS)
and Formaldehyde-Assisted Isolation of Regulatory Elements (FAIRE),
combined with these ChIP-seq assays for select
regulatory factors. DNaseI HS and FAIRE provide assay
cross-validation with commonly identified regions delineating the
highest confidence areas of open chromatin.  These ChIP assays provide
functional validation and preliminary annotation of a subset of
open chromatin sites. Each method employed Illumina (formerly Solexa)
sequencing by synthesis as the detection platform.
The Tier 1 and Tier 2 cell types were additionally verified by a
second platform, high-resolution 1% ENCODE tiled microarrays supplied by NimbleGen.
</P>
<P>As a background control experiment, the <B>input</B> genomic DNA sample that
was used for ChIP was sequenced. Crosslinked chromatin
was sheared and the crosslinks were reversed without carrying out the
immunoprecipitation step. This sample was otherwise processed in a manner
identical to the ChIP sample as described below. The input track is
useful in revealing potential artifacts arising from the sequence
alignment process such as copy number differences between the
reference genome and the sequenced samples, as well as regions of
poor sequence alignability.
</P>
<P>Other <B><em>Open Chromatin</em></B> track sets:
<UL>
<LI>Data for the DNaseI HS experiments can be found in
<A HREF="/cgi-bin/hgTrackUi?db=hg19&g=wgEncodeOpenChromDnase">Duke DNaseI HS</A>.
<LI>Data for the FAIRE experiments can be found in
<A HREF="/cgi-bin/hgTrackUi?db=hg19&g=wgEncodeOpenChromFaire">UNC FAIRE</A>.
<LI>A synthesis of all the open chromatin assays for select cell lines can
be found in
<A HREF="/cgi-bin/hgTrackUi?db=hg19&g=wgEncodeOpenChromSynth"
target=_blank>Open Chrom Synth</a>.
</UL>

<H2>Display Conventions and Configuration</H2>

<P>This track is a multi-view composite track that contains a single data type
with multiple levels of annotation (views). For each view, there are
multiple subtracks representing different cell types that display individually
on the browser. Instructions for configuring multi-view tracks are
<A HREF="/goldenPath/help/multiView.html" TARGET=_BLANK>here</A>.
ChIP data displayed here represents a continuum of signal intensities.
The Iyer lab recommends setting the &quot;Data view scaling: <EM>auto-scale</EM>&quot;
option when viewing signal data in full mode to see the <EM>full</EM> dynamic
range of the data. Note that in regions that do not have open chromatin sites,
autoscale will rescale the data and inflate the background signal, making the
regions appear noisy. Changing back to fixed scale will alleviate this issue.
In general, for each experiment in each of the cell types, the
<B>UTA TFBS</B> tracks contain the following views:
<DL>
<DT><I>Peaks</I></DT>
<DD>Regions of enriched signal in ChIP experiments.
Peaks were called based on signals created using F-Seq, a software program
developed at Duke (Boyle <em>et al.</em>, 2008b). Significant regions were
determined by fitting the data to a gamma distribution to calculate p-values.
Contiguous regions where p-values were below a 0.05/0.01 threshold were
considered significant.  The solid vertical line in the peak represents the
point with highest signal.</DD>
<DT><I>F-Seq Density Signal</I></DT>
<DD>Density graph (wiggle) of signal
enrichment calculated using F-Seq for the combined set of sequences from all
replicates. F-Seq employs Parzen kernel density estimation to create base pair
scores (Boyle <em>et al.</em>, 2008b). This method does not look at fixed-length
windows but rather weights contributions of nearby sequences in proportion to
their distance from that base.  It only considers sequences aligned 4 or less
times in the genome and uses an alignability background model to try to correct
for regions where sequences cannot be aligned. For each cell type, especially
important for those with an abnormal karyotype, a model to try to correct for
amplifications and deletions that is based on control input data was also used.</DD>
<DT><I>Base Overlap Signal</I></DT>
<DD>An alternative version of the
<I>F-Seq Density Signal</I> track annotation that provides a higher resolution
view of the raw sequence data. This track also includes the combined set of
sequences from all replicates.  For each sequence, the aligned read is
extended 5 bp in both directions from its 5' aligned end where DNase cut
the DNA.  The score at each base pair represents the number of
extended fragments that overlap the base pair.</DD>
</DL>
</P>
<P>Peaks and signals displayed in this track are the results of pooled replicates. The raw
sequence and alignment files for each replicate are available for
<A HREF="/cgi-bin/hgFileUi?g=wgEncodeOpenChromChip">download</A>.
</P>
<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>

<H2>Methods</H2>

<P>Cells were grown according to the approved
<A HREF="/ENCODE/protocols/cell" TARGET=_BLANK>ENCODE cell culture protocols</A>.
</P>
<P>To perform <em>ChIP</em>, proteins were cross-linked to DNA <em>in vivo</em>
using 1&#37; formaldehyde solution (Bhinge <em>et al.</em>, 2007; ENCODE
Project Consortium, 2007). Cross-linked chromatin was sheared by sonication
and immunoprecipitated using a specific antibody against the protein of
interest. After reversal of the cross-links, the immunoprecipitated DNA was
used to identify the genomic location of transcription factor binding. This
was accomplished by sequencing of the ends of the immunoprecipitated DNA
(ChIP-seq) using the Illumina (Solexa) sequencing system. ChIP data for
Tier 1 and Tier 2 cell lines were verified by comparing multiple independent
growths (replicates) and determining the reproducibility of the data. For
some cell types, additional verification was performed using the same
immunoprecipitated DNA by labeling and hybridizing to NimbleGen Human
ENCODE tiling arrays (1&#37; of the genome) along with the input DNA as reference
(ChIP-chip). A more detailed protocol is available
<A HREF="/ENCODE/protocols/general/UTA_ChIPseq_protocol.pdf">here</A>.
</P>
<P>DNA fragments isolated by ChIP are 100-200 bp in length, with the average
length being 134 bp.  Sequences from each experiment were aligned to the
genome using Burrows-Wheeler Aligner (Li <em>et al.</em>, 2010) for the GRCh37 (hg19) assembly.
<DIV style='position:relative; left:25px;'><DL><DT>The command used for these alignments was:</DT>
<DD><code>&gt; bwa aln -t 8 genome.fa s_1.sequence.txt.bfq &gt; s_1.sequence.txt.sai</code></DD>
<DT>Where genome.fa is the whole genome sequence and s_1.sequence.txt.bfq is one lane
of sequences converted into the required bfq format.</DT></DL></DIV>
</P>
<P>Sequences from multiple lanes
are combined for a single replicate using the bwa samse command, and converted
in the sam/bam format using SAMtools.
</P>
<P>Only those that aligned to 4 or fewer locations were retained. Other sequences
were also filtered based on their alignment to problematic regions
(such as satellites and rRNA genes - see
<A HREF="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeOpenChromChip/supplemental/"
 TARGET=ucscDownloads>supplemental materials</a>).
The mappings of these short reads to the genome are available for
<A HREF="/cgi-bin/hgFileUi?g=wgEncodeOpenChromChip">
download</A>.
</P>
<P>The resulting digital signal was converted to a continuous wiggle track using
F-Seq that employs Parzen kernel density estimation to create base pair scores
(Boyle <em>et al.</em>, 2008b).  Input data has been generated for several
cell lines. These are used directly to create a control/background model used
for F-Seq when generating signal annotations for these cell lines.
These models are meant to correct for sequencing biases, alignment artifacts,
and copy number changes in these cell lines. Input data is not being generated
directly for other cell lines. Instead, a general background model was derived
from the available input data sets. This should provide corrections for
sequencing biases and alignment artifacts, but will not correct for cell
type specific copy number changes.
<DIV style='position:relative; left:25px;'>
<DL><DT>The exact command used for this step is:</DT>
<DT>CTCF:</DT>
<DD><code>&gt; fseq -l 300 -v -b &lt;bff files&gt; -p &lt;iff files&gt; aligments.bed</code></DD>
<DT>c-Myc:</DT>
<DD><code>&gt; fseq -l 600 -v -b &lt;bff files&gt; -p &lt;iff files&gt; aligments.bed</code></DD>
<DT>PolII:</DT>
<DD><code>&gt; fseq -l 800 -v -b &lt;bff files&gt; -p &lt;iff files&gt; aligments.bed</code></DD>
<DT>Where the (bff files) are the background files based on alignability, the
(iff files) are the background files based on the input experiments,
and alignments.bed are a bed file of filtered sequence alignments.</DT></DL></DIV>
</P>
<P>Discrete ChIP sites (peaks) were identified from ChIP-seq F-seq
density signal.  Significant regions were determined by fitting the
data to a gamma distribution to calculate p-values. Contiguous regions
where p-values were below a 0.05/0.01 threshold were considered significant.
</P>
<P>Data from the high-resolution 1% ENCODE tiled microarrays supplied by
NimbleGen were normalized using the Tukey biweight normalization, and peaks
were called using ChIPOTle (Buck, <em>et al.</em>, 2005) at multiple levels
of significance. Regions matched on size to these peaks that were devoid of
any significant signal were also created as a null model.  These data were used
for additional verification of Tier 1 and Tier 2 cell lines by ROC analysis.
Files labeled Validation view containing this data are available for
<a href="/cgi-bin/hgFileUi?db=hg19&g=wgEncodeOpenChromChip" >download</a>.

</P>

<h2>Release Notes</h2>

<P>
Release 2 (August 2011) of this track adds 34 new experiments including 17 new cell lines.
</p>
<P>
<li><B>Enhancer and Insulator Functional assays:</B>  A subset of DNase and FAIRE
regions were cloned into functional tissue culture reporter assays to test for
enhancer and insulator activity.  Coordinates and results from these
experiments can be found
<A HREF="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeOpenChromChip/supplemental/"
 TARGET=ucscDownloads>here</a>.
</UL></P>

<H2>Credits</H2>

<P>
These data and annotations were created by a collaboration of multiple
institutions (contact:
<A HREF="mailto:&#116;&#115;&#102;u&#114;&#101;&#121;&#64;&#101;m&#97;&#105;&#108;.
&#117;&#110;&#99;.
&#101;&#100;u">
Terry Furey</A>)
<!-- above address is tsfurey at email.unc.edu -->
</P><P>
<UL>
<LI>Duke University's <A HREF="http://www.genome.duke.edu/index.php"
TARGET=_blank>Institute for Genome Sciences & Policy (IGSP)</A>: Alan Boyle,
Lingyun Song, and
<A HREF="http://www.genome.duke.edu/people/faculty/crawford/"
TARGET=_blank>Greg Crawford</a>
</P><P>
<LI>University of North Carolina at Chapel Hill: Paul Giresi and
<A HREF="http://bio.unc.edu/people/faculty/lieb/"
TARGET=_blank>Jason Lieb</a> and <A HREF="http://genetics.unc.edu/faculty/terry-furey"
TARGET=_blank>Terry Furey</A>
</P><P>
<LI>University of Texas at Austin: Zheng Liu, Ryan McDaniell, Bum-Kyu Lee, and
<A HREF="http://microarray.icmb.utexas.edu/research.html"
TARGET=_blank>Vishy Iyer</a>
</P><P>
<LI><A HREF="http://www.ebi.ac.uk/" TARGET=_blank>European Bioinformatics
Insitute</A>: Paul Flicek, Damian Keefe, and
<A HREF="http://www.ebi.ac.uk/~birney/" TARGET=_blank>Ewan Birney</a>
</P><P>
<LI><A HREF="http://www.oncology.cam.ac.uk/" TARGET=_blank>University of
Cambridge, Department of Oncology and CR-UK Cambridge Research Institute (CRI)
</A>: Stefan Graf
</UL>
</P><P>
We thank NHGRI for ENCODE funding support.
</P>

<H2>References</H2>

<p>
Bhinge AA, Kim J, Euskirchen GM, Snyder M, Iyer VR.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17568006" target="_blank">
Mapping the chromosomal targets of STAT1 by Sequence Tag Analysis of Genomic Enrichment (STAGE)</a>.
<em>Genome Res</em>. 2007 Jun;17(6):910-6.
</p>

<p>
Boyle AP, Davis S, Shulha HP, Meltzer P, Margulies EH, Weng Z, Furey TS, Crawford GE.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18243105" target="_blank">
High-resolution mapping and characterization of open chromatin across the genome</a>.
<em>Cell</em>. 2008 Jan 25;132(2):311-22.
</p>

<p>
Boyle AP, Guinney J, Crawford GE, Furey TS.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18784119" target="_blank">
F-Seq: a feature density estimator for high-throughput sequence tags</a>.
<em>Bioinformatics</em>. 2008 Nov 1;24(21):2537-8.
</p>

<p>
Buck MJ, Nobel AB, Lieb JD.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16277752" target="_blank">
ChIPOTle: a user-friendly tool for the analysis of ChIP-chip data</a>.
<em>Genome Biol</em>. 2005;6(11):R97.
</p>

<p>
Crawford GE, Davis S, Scacheri PC, Renaud G, Halawi MJ, Erdos MR, Green R, Meltzer PS, Wolfsberg TG, Collins FS.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16791207" target="_blank">
DNase-chip: a high-resolution method to identify DNase I hypersensitive sites using tiled microarrays</a>.
<em>Nat Methods</em>. 2006 Jul;3(7):503-9.
</p>

<p>
Crawford GE, Holt IE, Whittle J, Webb BD, Tai D, Davis S, Margulies EH, Chen Y, Bernat JA, Ginsburg D <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16344561" target="_blank">
Genome-wide mapping of DNase hypersensitive sites using massively parallel signature sequencing (MPSS)</a>.
<em>Genome Res</em>. 2006 Jan;16(1):123-31.
</p>

<p>
ENCODE Project Consortium, Birney E, Stamatoyannopoulos JA, Dutta A, Guig&#243; R, Gingeras TR, Margulies EH, Weng Z, Snyder M, Dermitzakis ET <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17571346" target="_blank">
Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</a>.
<em>Nature</em>. 2007 Jun 14;447(7146):799-816.
</p>

<p>
Giresi PG, Kim J, McDaniell RM, Iyer VR, Lieb JD.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/17179217" target="_blank">
FAIRE (Formaldehyde-Assisted Isolation of Regulatory Elements) isolates active regulatory elements from human chromatin</a>.
<em>Genome Res</em>. 2007 Jun;17(6):877-85.
</p>

<p>
Giresi PG, Lieb JD.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19303047" target="_blank">
Isolation of active regulatory elements from eukaryotic chromatin using FAIRE (Formaldehyde Assisted Isolation of Regulatory Elements)</a>.
<em>Methods</em>. 2009 Jul;48(3):233-9.
</p>

<p>
Lee BK, Bhinge AA, Battenhouse A, McDaniell RM, Liu Z, Song L, Ni Y, Birney E, Lieb JD, Furey TS <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/22090374" target="_blank">
Cell-type specific and combinatorial usage of diverse transcription factors revealed by genome-wide binding studies in multiple human cells</a>.
<em>Genome Res</em>. 2012 Jan;22(1):9-24.
</p>

<p>
Li H, Ruan J, Durbin R.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18714091" target="_blank">
Mapping short DNA sequencing reads and calling variants using mapping quality scores</a>.
<em>Genome Res</em>. 2008 Nov;18(11):1851-8.
</p>

<p>
Song L, Crawford GE.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20150147" target="_blank">
DNase-seq: a high-resolution technique for mapping active gene regulatory elements across the genome from mammalian cells</a>.
<em>Cold Spring Harb Protoc</em>. 2010 Feb;2010(2):pdb.prot5384.
</p>

<h2>Publications</h2>

<p>
Lee BK, Bhinge AA, Battenhouse A, McDaniell RM, Liu Z, Song L, Ni Y, Birney E, Lieb JD, Furey TS <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/22090374" target="_blank">
Cell-type specific and combinatorial usage of diverse transcription factors revealed by genome-wide binding studies in multiple human cells</a>.
<em>Genome Res</em>. 2012 Jan;22(1):9-24.
</p>

<H2>Data Release Policy</H2>
Data users may freely use ENCODE data, but may not, without prior consent,
submit publications that use an unpublished ENCODE dataset until nine months
following the release of the dataset.  This date is listed in the <EM>Restricted
Until</EM> column on the track configuration page and the download page.  The
full data release policy for ENCODE is available <A HREF="../ENCODE/terms.html"
TARGET=_BLANK>here</A>.
