<H2>Description</H2>

<h3>Rationale for the Mouse ENCODE project</h3>
Our knowledge of the function of genomic DNA sequences comes from three 
basic approaches. Genetics uses changes in behavior or structure of a cell or organism
in response to changes in DNA sequence to infer function of the altered sequence. 
Biochemical approaches monitor states of histone modification, binding of specific 
transcription factors, accessibility to DNases and other epigenetic features along 
genomic DNA. In general, these are associated with gene activity, but the precise 
relationships remain to be established. The third approach is evolutionary, using 
comparisons among homologous DNA sequences to find segments that are evolving 
more slowly or more rapidly than expected given the local rate of neutral change. These 
are inferred to be under negative or positive selection, respectively, and we interpret 
these as DNA sequences needed for a preserved (negative selection) or adaptive 
(positive selection) function. <p></p>

<p>The ENCODE project aims to discover all the DNA sequences associated with 
various epigenetic features, with the reasonable expectation that these will also be 
functional (best tested by genetic methods). However, it is not clear how to relate these 
results with those from evolutionary analyses. The mouse ENCODE project aims to 
make this connection explicitly and with a moderate breadth. Assays identical to those 
being used in the ENCODE project are performed in cell types in mouse that are similar 
or homologous to those studied in the human project. Thus we will be able to discover 
which epigenetic features are conserved between mouse and human, and we can 
examine the extent to which these overlap with the DNA sequences under negative 
selection. The contribution of DNA with a function preserved in mammals versus 
that with a function in only one species will be discovered. The results will have a 
significant impact on our understanding of the evolution of gene regulation.
</p>

<H3>Reference transcriptome measurements with RNA-seq</H3>
RNA-seq is a method for mapping and quantifying the transcriptome of any organism that has a genomic DNA sequence assembly (Mortazavi <EM>et al.</EM>, 2008).
RNA-seq is performed by reverse-transcribing an RNA sample into cDNA, followed by high-throughput DNA sequencing,
which was done here on the Illumina HiSeq sequencer. The transcriptome measurements shown on these tracks were performed on 
<A target="_BLANK" href="/cgi-bin/hgEncodeVocab?term=longPolyA&type=rnaExtract">polyA selected RNA</A> from 
<A target="_BLANK" href="/cgi-bin/hgEncodeVocab?term=cell&type=localization">total cellular RNA</A>. PolyA-selected RNA was fragmented 
by magnesium-catalyzed hydrolysis and then converted into cDNA by random priming and amplified. Paired-end 2x100 bp reads were obtained from each end of a cDNA fragment. 
Reads were aligned to the mm9 human reference genome using TopHat (Trapnell <em>et al.</em>, 2009), a program specifically designed to align RNA-seq reads and discover splice junctions de novo.
All sequence and alignments files are available on the <A target="_BLANK" href="/cgi-bin/hgFileUi?db=mm9&g=wgEncodeCaltechRnaSeq">downloads page</A>.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track is a multi-view composite track that contains multiple data types
(<EM>views</EM>).  For each view, there are multiple subtracks that
display individually on the browser.  Instructions for configuring multi-view
tracks are <A href="/goldenPath/help/multiView.html" target="_BLANK">here</A>.
The following views are in this track:
<DL>
<DT><I>Alignments</I></DT>
<DD>The Alignments (BAM file) view shows reads aligned to the genome. Alignments are colored by cell type. See the <a href="http://bowtie-bio.sourceforge.net/manual.shtml#sam-bowtie-output" target=_blank>Bowtie Manual</a> (Langmead <em>et al.</em>, 2009)
for information about the SAM Bowtie output (including other tags) and the 
<a href="http://samtools.sourceforge.net/SAM1.pdf#page=4" target=_blank>SAM Format Specification</a> for information on 
the SAM/BAM file format. </DD></DT>
<DT><I>Raw Signal</I></DT>
<DD>Density graph (wiggle) of signal enrichment based on a normalized aligned 
read density (Read Per Million, RPM). The RPM measure assists in visualizing the relative amount of a given transcript across multiple samples. This is used to display all reads in this track.</DD>
<DT><I>Signal (Unique Reads)</I></DT>
<DD>Density graph (wiggle) of signal enrichment based on processed data. This is used to display uniquely mapped reads in this track.</DD>
</DL>

<P>
<!--echo var="BROWSER_HOST" -->
Additional views are available on the Downloads page. 
</P>

<H2>Methods</H2>

<H3>Experimental Procedures</H3>

<P> 
Cells were grown according to the approved <a href="/ENCODE/protocols/cell/mouse" target="_BLANK">ENCODE cell culture protocols</a>. 
Cells were lysed in RLT buffer (Qiagen RNEasy kit), and processed on RNEasy midi columns according to the manufacturer's protocol, with the inclusion of the "on-column" DNAse digestion step to remove residual genomic DNA. A quantity of 75 µgs of total RNA was selected twice with oligo-dT beads (Dynal) according to the manufacturer's 
protocol to isolate mRNA from each of the preparations.  A quantity of 100 ngs of mRNA was 
then processed according to the protocol in Mortazavi <EM>et al.</EM> (2008), and prepared for sequencing on the Illumina GAIIx or HiSeq platforms 
 according to the protocol for the ChIP-Seq DNA genomic DNA kit (Illumina). Paired-end libraries were size-selected around 200 bp (fragment length).
 
Libraries were sequenced with the Illumina HiSeq according to the manufacturer's recommendations. Paired-end reads of 100 bp length were obtained

<H3>Data Processing and Analysis</H3>
 
<P>Reads were mapped to the reference mouse genome (version mm9 with or without the Y chromosome, depending on the sex of the cell line, 
and without the random chromosomes in all cases) using <A href="http://tophat.cbcb.umd.edu/" target="_BLANK">TopHat</A> (version 1.3.1).  TopHat was used with default settings with the exception of specifying an empirically determined mean inner-mate distance and supplying known ENSEMBL version 63 splice junctions.</P>

<H2>Credits</H2>

<P>Wold Group: Brian Williams, Georgi Marinov, Diane Trout, Lorian Schaeffer, Gordon Kwan, Katherine Fisher, Gilberto De Salvo, Ali Mortazavi, Henry Amrhein, Brandon King </P>

<P>
Contacts: 
<A href="mailto:georgi@caltech.edu">
Georgi Marinov</A> (data coordination/informatics/experimental),
<!-- above address is georgi at caltech.edu -->
<A href="mailto:diane@caltech.edu">
Diane Trout</A> (informatics) and
<!-- above address is diane at caltech.edu -->
<A href="mailto:bawilli_91125@yahoo.com">
Brian Williams</A> (experimental)
<!-- above address is bawilli_91125 at yahoo.com -->
</P>

<H2>References</H2>

<p>
Langmead B, Trapnell C, Pop M, Salzberg SL.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" target="_blank">
Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</a>.
<em>Genome Biol</em>. 2009;10(3):R25.
</p>

<p>
Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045" target="_blank">
Mapping and quantifying mammalian transcriptomes by RNA-Seq</a>.
<em>Nat Methods</em>. 2008 Jul;5(7):621-8.
</p>

<p>
Trapnell C, Pachter L, Salzberg SL.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" target="_blank">
TopHat: discovering splice junctions with RNA-Seq</a>.
<em>Bioinformatics</em>. 2009 May 1;25(9):1105-11.
</p>

<H2> Data Release Policy </H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A href="/ENCODE/terms.html" target="_BLANK">here</A>.</P>
