<H2>Description</H2>

<p>This track was produced as part of the mouse ENCODE Project.
This track shows RNA-seq measured genome-wide in mouse
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType"> tissues and cell lines</A>.
Poly-A selected mRNA was used as the source for transcriptome profiling of tissues and cell types 
that also had corresponding DNase I hypersensitive profiles.
</p>

<H2>Display Conventions and Configuration</H2>
<p>
This track is a multi-view composite track that contains multiple data types (<EM>views</EM>). 
For each view, there are multiple subtracks that display individually on the browser. Instructions 
for configuring multi-view tracks are <a href="/goldenPath/help/multiView.html" target="_blank">here</a>.
Color differences among the views are arbitrary and they provide a visual cue for distinguishing
between the different cell and tissue types.  This track contains the following views:
<DL>
<DT><I>Plus</I> and <I>Minus</I> <I>Signals</I><DD>These views display clusters of overlapping read mappings 
on the forward and reverse genomic strands.<DD> 
<DT><I>Signal</I></DT><DD>Density graph (wiggle) of signal enrichment based on processed data.</DD>
<DT><I>Alignments</I></DT><DD>Mappings of short 50-base single end reads to the genome.  See the <A HREF="http://samtools.sourceforge.net/SAM1.pdf" TARGET="_BLANK">SAM Format Specification</A> for more information on the SAM/BAM file format. <DD>
</DL>

<H2>Methods</H2>
<p>
Cells were grown according to the approved
<a href="/ENCODE/protocols/cell/mouse" target="_blank">ENCODE cell culture protocols</a>.
Fresh tissues were harvested from mice and stored until used for preparing total RNA samples.  The total RNA 
was used as starting material to select poly-A RNA and used for constructing SOLiD libraries according to the 
protocols supplied by the manufacturer.  All RNA samples were spiked in with NIST standards before libraries were constructed.
 The RNA-seq libraries were sequenced on ABI SOLiD sequencing platform as 50-base reads according to the manufacturer's recommendations.
<p>
Reads were aligned to the mm9 reference genome using ABI BioScope software version 1.2.1.  Colorspace FASTQ format files were created 
using Heng Li's solid2fastq.pl script version 0.1.4 (Li <EM>et al.</EM>, 2009a), representing 0, 1, 2, 3 color codes with the letters A, C, G, T respectively.  
 Signal files were created from the BAM (Li <EM>et al.</EM>, 2009b) alignments using BEDTools (Quinlan <EM>et al.</EM>, 2010). 
</p>

<h2>Release Notes</h2>

<p>
This is Release 1 (July 2012). It contains a total of 25 RNA-seq experiments.
</p>


<H2>Credits</H2>

<p>These data were generated by the UW ENCODE group. </p>
<p>Contact: <A HREF="mailto:&#115;&#117;l&#108;&#64;&#117;.w&#97;&#115;h&#105;&#110;&#103;&#116;&#111;n.&#101;&#100;&#117;">
Richard Sandstrom</A>
<!-- above address is sull at u.washington.edu -->
</p>

<H2>References</H2>

<p>
Li H, Durbin R.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" target="_blank">
Fast and accurate short read alignment with Burrows-Wheeler transform</a>.
<em>Bioinformatics</em>. 2009a Jul 15;25(14):1754-60.
</p>

<p>
Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome
Project Data Processing Subgroup.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" target="_blank">
The Sequence Alignment/Map format and SAMtools</a>.
<em>Bioinformatics</em>. 2009b Aug 15;25(16):2078-9.
</p>

<p>
Quinlan AR, Hall IM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" target="_blank">
BEDTools: a flexible suite of utilities for comparing genomic features</a>.
<em>Bioinformatics</em>. 2010 Mar 15;26(6):841-2.
</p>


<H2> Data Release Policy </H2>

<p>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET=_BLANK HREF="../ENCODE/terms.html" TARGET=_BLANK> here</A>.</p>

