<h2>Description</h2>

<p>This track was produced as part of the ENCODE Project.
This track shows genome-wide assessment of DNA replication timing in
<A TARGET=_BLANK HREF="/cgi-bin/hgEncodeVocab?type=cellType">cell lines</A>
using NimbleGen tiling CGH microarrays. Each experiment represents
the relative enrichment of early vs. late S-phase nascent strands in
a given cell line, with data represented as a loess-smoothed function
of individual timing values at probes spaced at even intervals across 
the genome. Regions with high values indicate domains of early replication where
initiaion occurs earlier in S-phase or early in a higher proportion of cells. 
</p>

<h2>Display Conventions and Configuration</h2>
<p>
<DT><I>Wavelet-smoothed Signal</I></DT>
<DD>Wavelet-smoothed of mean early/late S-phase ratios.</DD>
</p>

<p>Metadata for a particular subtrack can be found by clicking the down arrow in the list of subtracks.</p>


<h2>Methods</h2>

<p>
Cells were grown according to the approved
<a href="/ENCODE/protocols/cell/mouse" target="_blank">ENCODE cell culture protocols</a>.
Methods for replication timing profile creation and analysis are described in detail in 
Hiratani<em> et al.</em> (2008) and Ryba <em>et al.</em> (June 2011).
Methods for individual stages are summarized below:
</p>

<h3>Extraction protocol</h3>

<p>
Replication timing data were obtained by hybridizing early and late replication intermediates to NimbleGen oligonucleotide arrays. 
Replication intermediates were prepared from cells that were first pulse-labeled with BrdU and then sorted into 
early (1st half of S) and late (2nd half of S) stages of S-phase by flow cytometry, followed by anti-BrdU 
immunoprecipitation of the BrdU-substituted (nascent) replication intermediates that were synthesized either 
early or late during S-phase. Samples were labeled after unbiased amplification of recovered DNA by whole-genome amplification (WGA; Sigma, GenomePlex). 
</p>

<h3>Hybridization protocol</h3>

<p>
The hydridization set used the <a  href="http://www.nimblegen.com/products/lit/index.html" target=_blank> NimbleGen standard protocol</a>. 
Cy3- and Cy5-labeled DNA samples (6 &micro;g each) were co-hybridized to Nimblegen CGH arrays 
containing evenly-spaced oligonucleotide probes across the mouse genome, with a median probe spacing of 1.1-5.8 kb. 
No differences in smoothed data have been detected with probe densities from 100 bp to 5.8 kb. 
</p>

<h3>Scan protocol</h3>

<p>
NimbleGen MS 200 2 &micro;m resolution scanner and GenePix software were used per <a href="http://www.nimblegen.com/products/lit/index.html" target=_blank> NimbleGen's standard protocol</a>.
</p>

<h3>Data processing</h3>

<p>
NimbleScan software was used to obtain .pair raw data per manufacturer's instructions. 
Raw early/late data (i.e., from .pair files) from two independent biological replicates in 
which early- and late-replicating DNA were labeled reciprocally were loess-normalized to remove signal intensity-dependent bias,
scaled to a reference data set to have the same median absolute deviation and then averaged (limma package, R/Bioconductor). 
The mean early/late ratios were used to generate a smoothed profile (i.e., processed data) using local polynomial smoothing (loess, 300 kb span) 
for each chromosome using basic functions in the statistical language R. 
</p>


<h2>Verification</h2>

<p>
Technical data quality was assessed by verifying high auto-correlation between neighboring timing values. 
Biological identity was confirmed by verifying consistent early or late replication by PCR at individual loci, as well as uniformity in replication profiles between replicate experiments.
</p>

<h2>Credits</h2>

<p>These data were generated by the FSU ENCODE group. </p>
<p>Contact: <A HREF="mailto:&#103;&#105;&#108;b&#101;r&#116;&#64;&#98;&#105;&#111;.&#102;&#115;&#117;.&#101;du">
David M. Gilbert</A>
<!-- above address is gilbert at bio.fsu.edu -->
</p>

<h2>References</h2>

<p>
Hiratani I, Ryba T, Itoh M, Rathjen J, Kulik M, Papp B, Fussner E, Bazett-Jones DP, Plath K, Dalton S <em>et al</em>.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19952138"
target="_blank">
Genome-wide dynamics of replication timing revealed by in vitro models of mouse embryogenesis</a>.
<em>Genome Res</em>. 2010 Feb;20(2):155-69.
</p>

<p>
Hiratani I, Ryba T, Itoh M, Yokochi T, Schwaiger M, Chang CW, Lyou Y, Townes TM, Sch&#252;beler D, Gilbert DM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18842067"
target="_blank">
Global reorganization of replication domains during embryonic stem cell differentiation</a>.
<em>PLoS Biol</em>. 2008 Oct 7;6(10):e245.
</p>

<p>
Pope BD, Tsumagari K, Battaglia D, Ryba T, Hiratani I, Ehrlich M, Gilbert DM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/22096571"
target="_blank">
DNA replication timing is maintained genome-wide in primary human myoblasts independent of D4Z4 contraction in FSH muscular dystrophy</a>.
<em>PLoS One</em>. 2011;6(11):e27413.
</p>

<p>
Ryba T, Battaglia D, Pope BD, Hiratani I, Gilbert DM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21637205"
target="_blank">
Genome-scale analysis of replication timing: from bench to bioinformatics</a>.
<em>Nat Protoc</em>. 2011 Jun;6(6):870-95.
</p>

<p>
Ryba T, Hiratani I, Lu J, Itoh M, Kulik M, Zhang J, Schulz TC, Robins AJ, Dalton S, Gilbert DM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20430782"
target="_blank">
Evolutionarily conserved replication timing profiles predict long-range chromatin interactions and distinguish closely related cell types</a>.
<em>Genome Res</em>. 2010 Jun;20(6):761-70.
</p>

<p>
Ryba T, Hiratani I, Sasaki T, Battaglia D, Kulik M, Zhang J, Dalton S, Gilbert DM.
<a href="http://www.ncbi.nlm.nih.gov/pubmed/22028635"
target="_blank">
Replication timing: a fingerprint for cell identity and pluripotency</a>.
<em>PLoS Comput Biol</em>. 2011 Oct;7(10):e1002225.
</p>


<h2> Data Release Policy </h2>

<p>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the <EM>Restricted Until</EM> column, above.  The full data release policy
for ENCODE is available
<A TARGET=_BLANK HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</p>

